{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np # linear algebra\n",
    "import pandas as pd # data processing, CSV file I/O (e.g. pd.read_csv)\n",
    "import glob\n",
    "import json\n",
    "from pprint import pprint\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. Set up file path and Read Files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "GLOBAL_PATH=\"../data/CORD-19-research-challenge\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "cord_19_embeddings_df=pd.read_csv(GLOBAL_PATH+'/cord_19_embeddings_4_17/cord_19_embeddings_4_17.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>xqhn0vbp</th>\n",
       "      <th>0.2771094739437103</th>\n",
       "      <th>-2.9749464988708496</th>\n",
       "      <th>1.10358726978302</th>\n",
       "      <th>9.312440872192383</th>\n",
       "      <th>1.3013277053833008</th>\n",
       "      <th>-4.281131267547607</th>\n",
       "      <th>-0.5318851470947266</th>\n",
       "      <th>-4.497644424438477</th>\n",
       "      <th>-0.6967141628265381</th>\n",
       "      <th>...</th>\n",
       "      <th>-0.31924888491630554</th>\n",
       "      <th>4.262275695800781</th>\n",
       "      <th>2.845273971557617</th>\n",
       "      <th>-1.656341552734375</th>\n",
       "      <th>-2.5844340324401855</th>\n",
       "      <th>-4.678825855255127</th>\n",
       "      <th>3.175534248352051</th>\n",
       "      <th>4.8904242515563965</th>\n",
       "      <th>-1.488284707069397</th>\n",
       "      <th>-1.3013025522232056</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>gi6uaa83</td>\n",
       "      <td>-0.344760</td>\n",
       "      <td>-4.762074</td>\n",
       "      <td>3.647769</td>\n",
       "      <td>2.616938</td>\n",
       "      <td>3.856543</td>\n",
       "      <td>-0.460272</td>\n",
       "      <td>0.899565</td>\n",
       "      <td>2.284138</td>\n",
       "      <td>0.590927</td>\n",
       "      <td>...</td>\n",
       "      <td>-0.185362</td>\n",
       "      <td>0.603534</td>\n",
       "      <td>1.475987</td>\n",
       "      <td>1.804034</td>\n",
       "      <td>2.819780</td>\n",
       "      <td>-4.039684</td>\n",
       "      <td>-3.027682</td>\n",
       "      <td>0.997251</td>\n",
       "      <td>-0.661524</td>\n",
       "      <td>1.590745</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>le0ogx1s</td>\n",
       "      <td>-3.257521</td>\n",
       "      <td>-1.720686</td>\n",
       "      <td>1.438893</td>\n",
       "      <td>-1.614458</td>\n",
       "      <td>-3.205178</td>\n",
       "      <td>1.125262</td>\n",
       "      <td>-1.420612</td>\n",
       "      <td>-4.270585</td>\n",
       "      <td>-2.557855</td>\n",
       "      <td>...</td>\n",
       "      <td>-1.691883</td>\n",
       "      <td>0.594955</td>\n",
       "      <td>1.336509</td>\n",
       "      <td>0.036567</td>\n",
       "      <td>-2.065922</td>\n",
       "      <td>0.360089</td>\n",
       "      <td>0.832659</td>\n",
       "      <td>2.954574</td>\n",
       "      <td>-4.601091</td>\n",
       "      <td>-1.097578</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>fy4w7xz8</td>\n",
       "      <td>0.920366</td>\n",
       "      <td>-3.546179</td>\n",
       "      <td>-2.537739</td>\n",
       "      <td>6.372102</td>\n",
       "      <td>0.025263</td>\n",
       "      <td>-1.991629</td>\n",
       "      <td>-0.612892</td>\n",
       "      <td>0.983194</td>\n",
       "      <td>-2.316126</td>\n",
       "      <td>...</td>\n",
       "      <td>-0.412801</td>\n",
       "      <td>0.831824</td>\n",
       "      <td>2.101388</td>\n",
       "      <td>3.283052</td>\n",
       "      <td>-3.075248</td>\n",
       "      <td>-4.381618</td>\n",
       "      <td>1.925288</td>\n",
       "      <td>4.607265</td>\n",
       "      <td>-1.254582</td>\n",
       "      <td>-1.320425</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>0qaoam29</td>\n",
       "      <td>-1.145982</td>\n",
       "      <td>-5.231421</td>\n",
       "      <td>-1.958305</td>\n",
       "      <td>5.035599</td>\n",
       "      <td>-1.601498</td>\n",
       "      <td>-0.756577</td>\n",
       "      <td>1.757929</td>\n",
       "      <td>1.490937</td>\n",
       "      <td>0.192156</td>\n",
       "      <td>...</td>\n",
       "      <td>0.131644</td>\n",
       "      <td>3.836460</td>\n",
       "      <td>1.115987</td>\n",
       "      <td>1.524282</td>\n",
       "      <td>-2.017589</td>\n",
       "      <td>-5.102140</td>\n",
       "      <td>3.864315</td>\n",
       "      <td>1.679577</td>\n",
       "      <td>1.871223</td>\n",
       "      <td>-0.295825</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>qj4dh6rg</td>\n",
       "      <td>-1.728899</td>\n",
       "      <td>-5.419925</td>\n",
       "      <td>-0.482808</td>\n",
       "      <td>9.170734</td>\n",
       "      <td>-1.273682</td>\n",
       "      <td>-1.007223</td>\n",
       "      <td>1.836062</td>\n",
       "      <td>3.512957</td>\n",
       "      <td>1.623566</td>\n",
       "      <td>...</td>\n",
       "      <td>-0.164270</td>\n",
       "      <td>4.492704</td>\n",
       "      <td>1.499324</td>\n",
       "      <td>4.371634</td>\n",
       "      <td>-2.533002</td>\n",
       "      <td>-4.532452</td>\n",
       "      <td>2.206654</td>\n",
       "      <td>1.812408</td>\n",
       "      <td>0.771027</td>\n",
       "      <td>-1.834923</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows Ã— 769 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   xqhn0vbp  0.2771094739437103  -2.9749464988708496  1.10358726978302  \\\n",
       "0  gi6uaa83           -0.344760            -4.762074          3.647769   \n",
       "1  le0ogx1s           -3.257521            -1.720686          1.438893   \n",
       "2  fy4w7xz8            0.920366            -3.546179         -2.537739   \n",
       "3  0qaoam29           -1.145982            -5.231421         -1.958305   \n",
       "4  qj4dh6rg           -1.728899            -5.419925         -0.482808   \n",
       "\n",
       "   9.312440872192383  1.3013277053833008  -4.281131267547607  \\\n",
       "0           2.616938            3.856543           -0.460272   \n",
       "1          -1.614458           -3.205178            1.125262   \n",
       "2           6.372102            0.025263           -1.991629   \n",
       "3           5.035599           -1.601498           -0.756577   \n",
       "4           9.170734           -1.273682           -1.007223   \n",
       "\n",
       "   -0.5318851470947266  -4.497644424438477  -0.6967141628265381  ...  \\\n",
       "0             0.899565            2.284138             0.590927  ...   \n",
       "1            -1.420612           -4.270585            -2.557855  ...   \n",
       "2            -0.612892            0.983194            -2.316126  ...   \n",
       "3             1.757929            1.490937             0.192156  ...   \n",
       "4             1.836062            3.512957             1.623566  ...   \n",
       "\n",
       "   -0.31924888491630554  4.262275695800781  2.845273971557617  \\\n",
       "0             -0.185362           0.603534           1.475987   \n",
       "1             -1.691883           0.594955           1.336509   \n",
       "2             -0.412801           0.831824           2.101388   \n",
       "3              0.131644           3.836460           1.115987   \n",
       "4             -0.164270           4.492704           1.499324   \n",
       "\n",
       "   -1.656341552734375  -2.5844340324401855  -4.678825855255127  \\\n",
       "0            1.804034             2.819780           -4.039684   \n",
       "1            0.036567            -2.065922            0.360089   \n",
       "2            3.283052            -3.075248           -4.381618   \n",
       "3            1.524282            -2.017589           -5.102140   \n",
       "4            4.371634            -2.533002           -4.532452   \n",
       "\n",
       "   3.175534248352051  4.8904242515563965  -1.488284707069397  \\\n",
       "0          -3.027682            0.997251           -0.661524   \n",
       "1           0.832659            2.954574           -4.601091   \n",
       "2           1.925288            4.607265           -1.254582   \n",
       "3           3.864315            1.679577            1.871223   \n",
       "4           2.206654            1.812408            0.771027   \n",
       "\n",
       "   -1.3013025522232056  \n",
       "0             1.590745  \n",
       "1            -1.097578  \n",
       "2            -1.320425  \n",
       "3            -0.295825  \n",
       "4            -1.834923  \n",
       "\n",
       "[5 rows x 769 columns]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cord_19_embeddings_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "metadata_path = f'{GLOBAL_PATH}/metadata.csv'\n",
    "meta_df = pd.read_csv(metadata_path, dtype={\n",
    "    'pubmed_id': str,\n",
    "    'Microsoft Academic Paper ID': str, \n",
    "    'doi': str\n",
    "})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>cord_uid</th>\n",
       "      <th>sha</th>\n",
       "      <th>source_x</th>\n",
       "      <th>title</th>\n",
       "      <th>doi</th>\n",
       "      <th>pmcid</th>\n",
       "      <th>pubmed_id</th>\n",
       "      <th>license</th>\n",
       "      <th>abstract</th>\n",
       "      <th>publish_time</th>\n",
       "      <th>authors</th>\n",
       "      <th>journal</th>\n",
       "      <th>Microsoft Academic Paper ID</th>\n",
       "      <th>WHO #Covidence</th>\n",
       "      <th>has_pdf_parse</th>\n",
       "      <th>has_pmc_xml_parse</th>\n",
       "      <th>full_text_file</th>\n",
       "      <th>url</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>zjufx4fo</td>\n",
       "      <td>b2897e1277f56641193a6db73825f707eed3e4c9</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Sequence requirements for RNA strand transfer ...</td>\n",
       "      <td>10.1093/emboj/20.24.7220</td>\n",
       "      <td>PMC125340</td>\n",
       "      <td>11742998</td>\n",
       "      <td>unk</td>\n",
       "      <td>Nidovirus subgenomic mRNAs contain a leader se...</td>\n",
       "      <td>2001-12-17</td>\n",
       "      <td>Pasternak, Alexander O.; van den Born, Erwin; ...</td>\n",
       "      <td>The EMBO Journal</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>http://europepmc.org/articles/pmc125340?pdf=re...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ymceytj3</td>\n",
       "      <td>e3d0d482ebd9a8ba81c254cc433f314142e72174</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Crystal structure of murine sCEACAM1a[1,4]: a ...</td>\n",
       "      <td>10.1093/emboj/21.9.2076</td>\n",
       "      <td>PMC125375</td>\n",
       "      <td>11980704</td>\n",
       "      <td>unk</td>\n",
       "      <td>CEACAM1 is a member of the carcinoembryonic an...</td>\n",
       "      <td>2002-05-01</td>\n",
       "      <td>Tan, Kemin; Zelus, Bruce D.; Meijers, Rob; Liu...</td>\n",
       "      <td>The EMBO Journal</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>http://europepmc.org/articles/pmc125375?pdf=re...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>wzj2glte</td>\n",
       "      <td>00b1d99e70f779eb4ede50059db469c65e8c1469</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Synthesis of a novel hepatitis C virus protein...</td>\n",
       "      <td>10.1093/emboj/20.14.3840</td>\n",
       "      <td>PMC125543</td>\n",
       "      <td>11447125</td>\n",
       "      <td>no-cc</td>\n",
       "      <td>Hepatitis C virus (HCV) is an important human ...</td>\n",
       "      <td>2001-07-16</td>\n",
       "      <td>Xu, Zhenming; Choi, Jinah; Yen, T.S.Benedict; ...</td>\n",
       "      <td>EMBO J</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>2sfqsfm1</td>\n",
       "      <td>cf584e00f637cbd8f1bb35f3f09f5ed07b71aeb0</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Structure of coronavirus main proteinase revea...</td>\n",
       "      <td>10.1093/emboj/cdf327</td>\n",
       "      <td>PMC126080</td>\n",
       "      <td>12093723</td>\n",
       "      <td>unk</td>\n",
       "      <td>The key enzyme in coronavirus polyprotein proc...</td>\n",
       "      <td>2002-07-01</td>\n",
       "      <td>Anand, Kanchan; Palm, Gottfried J.; Mesters, J...</td>\n",
       "      <td>The EMBO Journal</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>http://europepmc.org/articles/pmc126080?pdf=re...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>i0zym7iq</td>\n",
       "      <td>dde02f11923815e6a16a31dd6298c46b109c5dfa</td>\n",
       "      <td>PMC</td>\n",
       "      <td>Discontinuous and non-discontinuous subgenomic...</td>\n",
       "      <td>10.1093/emboj/cdf635</td>\n",
       "      <td>PMC136939</td>\n",
       "      <td>12456663</td>\n",
       "      <td>unk</td>\n",
       "      <td>Arteri-, corona-, toro- and roniviruses are ev...</td>\n",
       "      <td>2002-12-01</td>\n",
       "      <td>van Vliet, A.L.W.; Smits, S.L.; Rottier, P.J.M...</td>\n",
       "      <td>The EMBO Journal</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>True</td>\n",
       "      <td>True</td>\n",
       "      <td>custom_license</td>\n",
       "      <td>http://europepmc.org/articles/pmc136939?pdf=re...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   cord_uid                                       sha source_x  \\\n",
       "0  zjufx4fo  b2897e1277f56641193a6db73825f707eed3e4c9      PMC   \n",
       "1  ymceytj3  e3d0d482ebd9a8ba81c254cc433f314142e72174      PMC   \n",
       "2  wzj2glte  00b1d99e70f779eb4ede50059db469c65e8c1469      PMC   \n",
       "3  2sfqsfm1  cf584e00f637cbd8f1bb35f3f09f5ed07b71aeb0      PMC   \n",
       "4  i0zym7iq  dde02f11923815e6a16a31dd6298c46b109c5dfa      PMC   \n",
       "\n",
       "                                               title  \\\n",
       "0  Sequence requirements for RNA strand transfer ...   \n",
       "1  Crystal structure of murine sCEACAM1a[1,4]: a ...   \n",
       "2  Synthesis of a novel hepatitis C virus protein...   \n",
       "3  Structure of coronavirus main proteinase revea...   \n",
       "4  Discontinuous and non-discontinuous subgenomic...   \n",
       "\n",
       "                        doi      pmcid pubmed_id license  \\\n",
       "0  10.1093/emboj/20.24.7220  PMC125340  11742998     unk   \n",
       "1   10.1093/emboj/21.9.2076  PMC125375  11980704     unk   \n",
       "2  10.1093/emboj/20.14.3840  PMC125543  11447125   no-cc   \n",
       "3      10.1093/emboj/cdf327  PMC126080  12093723     unk   \n",
       "4      10.1093/emboj/cdf635  PMC136939  12456663     unk   \n",
       "\n",
       "                                            abstract publish_time  \\\n",
       "0  Nidovirus subgenomic mRNAs contain a leader se...   2001-12-17   \n",
       "1  CEACAM1 is a member of the carcinoembryonic an...   2002-05-01   \n",
       "2  Hepatitis C virus (HCV) is an important human ...   2001-07-16   \n",
       "3  The key enzyme in coronavirus polyprotein proc...   2002-07-01   \n",
       "4  Arteri-, corona-, toro- and roniviruses are ev...   2002-12-01   \n",
       "\n",
       "                                             authors           journal  \\\n",
       "0  Pasternak, Alexander O.; van den Born, Erwin; ...  The EMBO Journal   \n",
       "1  Tan, Kemin; Zelus, Bruce D.; Meijers, Rob; Liu...  The EMBO Journal   \n",
       "2  Xu, Zhenming; Choi, Jinah; Yen, T.S.Benedict; ...            EMBO J   \n",
       "3  Anand, Kanchan; Palm, Gottfried J.; Mesters, J...  The EMBO Journal   \n",
       "4  van Vliet, A.L.W.; Smits, S.L.; Rottier, P.J.M...  The EMBO Journal   \n",
       "\n",
       "  Microsoft Academic Paper ID WHO #Covidence  has_pdf_parse  \\\n",
       "0                         NaN            NaN           True   \n",
       "1                         NaN            NaN           True   \n",
       "2                         NaN            NaN           True   \n",
       "3                         NaN            NaN           True   \n",
       "4                         NaN            NaN           True   \n",
       "\n",
       "   has_pmc_xml_parse  full_text_file  \\\n",
       "0               True  custom_license   \n",
       "1               True  custom_license   \n",
       "2               True  custom_license   \n",
       "3               True  custom_license   \n",
       "4               True  custom_license   \n",
       "\n",
       "                                                 url  \n",
       "0  http://europepmc.org/articles/pmc125340?pdf=re...  \n",
       "1  http://europepmc.org/articles/pmc125375?pdf=re...  \n",
       "2  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...  \n",
       "3  http://europepmc.org/articles/pmc126080?pdf=re...  \n",
       "4  http://europepmc.org/articles/pmc136939?pdf=re...  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "meta_df.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2. Read BioXiv paper for testing training generator functions "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "biorxiv = '/biorxiv_medrxiv'\n",
    "bioxiv_json = glob.glob(f'{GLOBAL_PATH+biorxiv}/**/*.json', recursive=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'../data/CORD-19-research-challenge/biorxiv_medrxiv/biorxiv_medrxiv/pdf_json/f6b29be971089bfe0916c64ab9fbddcec38a7436.json'"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bioxiv_json[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "There are:  2278  pdf Json in BioXiv\n",
      "===================================================\n",
      "Dictionary keys: dict_keys(['paper_id', 'metadata', 'abstract', 'body_text', 'bib_entries', 'ref_entries', 'back_matter'])\n"
     ]
    }
   ],
   "source": [
    "print(\"There are: \",len(bioxiv_json),\" pdf Json in BioXiv\")\n",
    "print(\"===================================================\")\n",
    "all_files = []\n",
    "for filename in bioxiv_json:\n",
    "    #filename = biorxiv_dir + filename\n",
    "    file = json.load(open(filename, 'rb'))\n",
    "    all_files.append(file)\n",
    "\n",
    "file = all_files[0]\n",
    "print(\"Dictionary keys:\", file.keys())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'cite_spans': [],\n",
      "  'ref_spans': [],\n",
      "  'section': 'Abstract',\n",
      "  'text': 'Aims: To determine analytical capabilities of a commonly used '\n",
      "          'faecal immunochemical test (FIT) to detect haemoglobin (Hb) in the '\n",
      "          'context of NICE guidance DG30, and the likely use of FIT to '\n",
      "          'reprioritise patients delayed by the COVID-19 pandemic.'},\n",
      " {'cite_spans': [],\n",
      "  'ref_spans': [],\n",
      "  'section': 'Abstract',\n",
      "  'text': 'Methods: Data obtained from independent verification studies and '\n",
      "          'clinical testing of the HM-JACKarc FIT method in routine primary '\n",
      "          'care practice were analysed to derive analytical performance '\n",
      "          'characteristics.'},\n",
      " {'cite_spans': [],\n",
      "  'ref_spans': [],\n",
      "  'section': 'Abstract',\n",
      "  'text': 'Results: Detection capabilities for the FIT method were 0.5 Âµg/g '\n",
      "          '(limit of blank), 1.1 (limit of detection) and 15.0 Âµg/g (limit of '\n",
      "          'quantification). 31 of 33 (94%) non-homogenised specimens analysed '\n",
      "          'in triplicate were consistently categorised relative to 10 Âµg/g '\n",
      "          'compared to all 33 (100%) homogenised specimens. Imprecision in '\n",
      "          'non-homogenised specimens was higher (median 27.8%, (range 20.5% '\n",
      "          '-48.6%)) than in homogenised specimens (10.'}]\n"
     ]
    }
   ],
   "source": [
    "pprint(file['abstract'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "body_text content:\n",
      "[{'cite_spans': [{...}, {...}, {...}],\n",
      "  'ref_spans': [],\n",
      "  'section': 'Introduction',\n",
      "  'text': 'Colorectal cancer is globally the third most incident malignancy '\n",
      "          '(1) . It is surgically treatable with improved long-term outcomes '\n",
      "          'if diagnosis is at an early stage (2) . Most developed countries, '\n",
      "          'including the UK, operate colorectal screening programmes using '\n",
      "          'faecal occult blood testing. Screen detected cancers benefit from '\n",
      "          'early diagnosis and treatment, with associated improved survival '\n",
      "          '(2) . Faecal immunochemical tests (FIT) have largely replaced the '\n",
      "          'traditional guaiac based faecal occult blood tests due to the '\n",
      "          'increased specificity of FIT.'},\n",
      " {'cite_spans': [{...}, {...}],\n",
      "  'ref_spans': [],\n",
      "  'section': 'Introduction',\n",
      "  'text': 'To complement the UK Bowel Cancer Screening Programme (UKBCSP), the '\n",
      "          '2017 DG30 NICE guidance (3) recommended the use of FIT for faecal '\n",
      "          'haemoglobin (Hb) detection in patients presenting to primary care '\n",
      "          'with low risk abdominal symptoms. The adoption of FIT in primary '\n",
      "          'care has been slow, with notable variation in uptake and '\n",
      "          'implementation across the UK (4). The Oxford University Hospitals '\n",
      "          'Trust (OUH) adopted FIT prior to the DG30 guidance to comply with '\n",
      "          'the 2015 NG12 NICE guidance for suspected cancer (6) , which '\n",
      "          'recommended the use of faecal occult blood testing in symptomatic '\n",
      "          'patients. This coincided with a desire from the clinical laboratory '\n",
      "          'to move away from the comparatively inaccurate guaiac based method. '\n",
      "          'FIT was commissioned by Oxfordshire Clinical Commissioning Group '\n",
      "          '(OCCG) as a direct access test for General Practitioners in 2016 '\n",
      "          'following a study by this group comparing the accuracy of the '\n",
      "          'guaiac and FIT methods in symptomatic primary care patients meeting '\n",
      "          'NG12 criteria (5) .'}]\n"
     ]
    }
   ],
   "source": [
    "print(\"body_text content:\")\n",
    "pprint(file['body_text'][:2], depth=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['Introduction',\n",
      " 'Setting and analytical method',\n",
      " 'Method detection capability estimates and immunoassay reproducibility',\n",
      " 'Effects of sample homogenisation',\n",
      " 'Stability of specimens within the collection device',\n",
      " 'Within patient serial sampling',\n",
      " 'Statistical analysis',\n",
      " 'Detection capabilities and immunoassay reproducibility',\n",
      " 'Discussion',\n",
      " 'Implications for research and practice.',\n",
      " 'Concluding remarks.']\n"
     ]
    }
   ],
   "source": [
    "texts = [(di['section'], di['text']) for di in file['body_text']]\n",
    "texts_di = {di['section']: \"\" for di in file['body_text']}\n",
    "for section, text in texts:\n",
    "    texts_di[section] += text\n",
    "\n",
    "pprint(list(texts_di.keys()))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Introduction\n",
      "\n",
      "Colorectal cancer is globally the third most incident malignancy (1) . It is surgically treatable with improved long-term outcomes if diagnosis is at an early stage (2) . Most developed countries, including the UK, operate colorectal screening programmes using faecal occult blood testing. Screen detected cancers benefit from early diagnosis and treatment, with associated improved survival (2) . Faecal immunochemical tests (FIT) have largely replaced the traditional guaiac based faecal occult blood tests due to the increased specificity of FIT.To complement the UK Bowel Cancer Screening Programme (UKBCSP), the 2017 DG30 NICE guidance (3) recommended the use of FIT for faecal haemoglobin (Hb) detection in patients presenting to primary care with low risk abdominal symptoms. The adoption of FIT in primary care has been slow, with notable variation in uptake and implementation across the UK (4). The Oxford University Hospitals Trust (OUH) adopted FIT prior to the DG30 guidance to comply with the 2015 NG12 NICE guidance for suspected cancer (6) , which recommended the use of faecal occult blood testing in symptomatic patients. This coincided with a desire from the clinical laboratory to move away from the comparatively inaccurate guaiac based method. FIT was commissioned by Oxfordshire Clinical Commissioning Group (OCCG) as a direct access test for General Practitioners in 2016 following a study by this group comparing the accuracy of the guaiac and FIT methods in symptomatic primary care patients meeting NG12 criteria (5) .Despite gradual uptake of FIT, there remains a clear need to understand FIT method characteristics (7) . FIT testing, whether undertaken within a screening programme or applied to a symptomatic population, is dependent on the analytical performance of the laboratory procedures used. These analytical characteristics, most crucially sampling, impact on the way the results are most accurately reported (8) . Whilst there has been significant work undertaken in the context of screening (9) there is less data available in the symptomatic population. This is important to explore as the characteristics of the population for a screening programme, and the associated specimens, may differ from patients presenting with symptoms. For example, the age-range of the symptomatic population is broader and specimen characteristics, including faecal consistency, will likely differ in patients with changes in bowel habit, and the information and support for taking a sample is less. This may affect sample, and sampling integrity.The FIT methods used to process samples from symptomatic patients are those developed for measuring faecal Hb at higher concentrations in screening. At present, the cut-off used by the UKBCSP is >120 Âµg/g, with future plans to reduce this to approximately 50 Âµg/g to improve sensitivity. The threshold recommended for use with symptomatic patients in DG30 is 10 Âµg/g. FIT methods may not be optimised, or fully charac\n"
     ]
    }
   ],
   "source": [
    "body = \"\"\n",
    "\n",
    "for section, text in texts_di.items():\n",
    "    body += section\n",
    "    body += \"\\n\\n\"\n",
    "    body += text\n",
    "    body += \"\\n\\n\"\n",
    "\n",
    "print(body[:3000])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3. Flatten the Json file to create the training set "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "f6b29be971089bfe0916c64ab9fbddcec38a7436: Aims: To determine analytical capabilities of a commonly used faecal immunochemical test (FIT) to detect haemoglobin (Hb) in the context of NICE guidance DG30, and the likely use of FIT to reprioritis... Colorectal cancer is globally the third most incident malignancy (1) . It is surgically treatable with improved long-term outcomes if diagnosis is at an early stage (2) . Most developed countries, inc...\n"
     ]
    }
   ],
   "source": [
    "class FileReader:\n",
    "    def __init__(self, file_path):\n",
    "        with open(file_path) as file:\n",
    "            content = json.load(file)\n",
    "            self.paper_id = content['paper_id']\n",
    "            self.abstract = []\n",
    "            self.body_text = []\n",
    "            # Abstract\n",
    "            for entry in content['abstract']:\n",
    "                self.abstract.append(entry['text'])\n",
    "            # Body text\n",
    "            for entry in content['body_text']:\n",
    "                self.body_text.append(entry['text'])\n",
    "            self.abstract = '\\n'.join(self.abstract)\n",
    "            self.body_text = '\\n'.join(self.body_text)\n",
    "    def __repr__(self):\n",
    "        return f'{self.paper_id}: {self.abstract[:200]}... {self.body_text[:200]}...'\n",
    "\n",
    "first_row = FileReader(bioxiv_json[0])\n",
    "print(first_row)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing index: 0 of 2278\n",
      "Processing index: 227 of 2278\n",
      "Processing index: 454 of 2278\n",
      "Processing index: 681 of 2278\n",
      "Processing index: 908 of 2278\n",
      "Processing index: 1135 of 2278\n",
      "Processing index: 1362 of 2278\n",
      "Processing index: 1589 of 2278\n",
      "Processing index: 1816 of 2278\n",
      "Processing index: 2043 of 2278\n",
      "Processing index: 2270 of 2278\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paper_id</th>\n",
       "      <th>abstract</th>\n",
       "      <th>body_text</th>\n",
       "      <th>title</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>f6b29be971089bfe0916c64ab9fbddcec38a7436</td>\n",
       "      <td>Aims: To determine analytical capabilities of ...</td>\n",
       "      <td>Colorectal cancer is globally the third most i...</td>\n",
       "      <td>37232    Faecal immunochemical testing (FIT): ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>005d189d5bd7ac01aee65e934fd3d5186a3f7b27</td>\n",
       "      <td>The rapid outbreak of the new Coronavirus pand...</td>\n",
       "      <td>The outbreak of infectious diseases has always...</td>\n",
       "      <td>36952    Relationship between Average Daily Te...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>f905f78b32f63c6d14a79984dfb33f1b358b8ab4</td>\n",
       "      <td>New anti-AIDS treatments must be continually d...</td>\n",
       "      <td>In the absence of a curative treatment, the hi...</td>\n",
       "      <td>37478    Multimerization of HIV-1 integrase hi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>607e0074d8ff40c272b958c2fe48793fedfc785e</td>\n",
       "      <td></td>\n",
       "      <td>the author/funder, who has granted medRxiv a l...</td>\n",
       "      <td>36290    Virus shedding patterns in nasopharyn...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>72e25b728c6c62fb3a4e2c59c8ee48de4b5ee452</td>\n",
       "      <td>Recently classified as a pandemic by WHO, nove...</td>\n",
       "      <td>The pandemic of COVID-19 is taking a troll in ...</td>\n",
       "      <td>37135    Phylogenetic Analysis of the Novel Co...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                   paper_id  \\\n",
       "0  f6b29be971089bfe0916c64ab9fbddcec38a7436   \n",
       "1  005d189d5bd7ac01aee65e934fd3d5186a3f7b27   \n",
       "2  f905f78b32f63c6d14a79984dfb33f1b358b8ab4   \n",
       "3  607e0074d8ff40c272b958c2fe48793fedfc785e   \n",
       "4  72e25b728c6c62fb3a4e2c59c8ee48de4b5ee452   \n",
       "\n",
       "                                            abstract  \\\n",
       "0  Aims: To determine analytical capabilities of ...   \n",
       "1  The rapid outbreak of the new Coronavirus pand...   \n",
       "2  New anti-AIDS treatments must be continually d...   \n",
       "3                                                      \n",
       "4  Recently classified as a pandemic by WHO, nove...   \n",
       "\n",
       "                                           body_text  \\\n",
       "0  Colorectal cancer is globally the third most i...   \n",
       "1  The outbreak of infectious diseases has always...   \n",
       "2  In the absence of a curative treatment, the hi...   \n",
       "3  the author/funder, who has granted medRxiv a l...   \n",
       "4  The pandemic of COVID-19 is taking a troll in ...   \n",
       "\n",
       "                                               title  \n",
       "0  37232    Faecal immunochemical testing (FIT): ...  \n",
       "1  36952    Relationship between Average Daily Te...  \n",
       "2  37478    Multimerization of HIV-1 integrase hi...  \n",
       "3  36290    Virus shedding patterns in nasopharyn...  \n",
       "4  37135    Phylogenetic Analysis of the Novel Co...  "
      ]
     },
     "execution_count": 49,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "dict_ = {'paper_id': [], 'abstract': [], 'body_text': [], 'title': []}\n",
    "for idx, entry in enumerate(bioxiv_json):\n",
    "    if idx % (len(bioxiv_json) // 10) == 0:\n",
    "        print(f'Processing index: {idx} of {len(bioxiv_json)}')\n",
    "    \n",
    "    try:\n",
    "        content = FileReader(entry)\n",
    "    except Exception as e:\n",
    "        continue  # invalid paper format, skip\n",
    "    \n",
    "    # get metadata information\n",
    "    meta_data = meta_df.loc[meta_df['sha'] == content.paper_id]\n",
    "    # no metadata, skip this paper\n",
    "    if len(meta_data) == 0:\n",
    "        continue\n",
    "    \n",
    "    dict_['abstract'].append(content.abstract)\n",
    "    dict_['paper_id'].append(content.paper_id)\n",
    "    dict_['body_text'].append(content.body_text)\n",
    "    \n",
    "    # get metadata information\n",
    "    meta_data = meta_df.loc[meta_df['sha'] == content.paper_id]\n",
    "    \n",
    "    \n",
    "    # add the title information, add breaks when needed\n",
    "    dict_['title'].append(meta_data['title'])\n",
    "\n",
    "    \n",
    "    \n",
    "df_covid = pd.DataFrame(dict_, columns=['paper_id', 'abstract', 'body_text', 'title'])\n",
    "df_covid.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "2278"
      ]
     },
     "execution_count": 60,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(df_covid[\"body_text\"])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4. Cleaning the Text "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [],
   "source": [
    "import nltk\n",
    "from nltk.tokenize import sent_tokenize\n",
    "import sentencepiece as spm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "regex_tokenizer = nltk.RegexpTokenizer(\"\\w+\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [],
   "source": [
    "def normalize_text(text):\n",
    "    # lowercase text\n",
    "    text = str(text).lower()\n",
    "    # remove non-UTF\n",
    "    text = text.encode(\"utf-8\", \"ignore\").decode()\n",
    "    # remove punktuation symbols\n",
    "    text = \" \".join(regex_tokenizer.tokenize(text))\n",
    "    return text\n",
    "\n",
    "def sentencise_text(text):\n",
    "    text = sent_tokenize(text)\n",
    "    return text \n",
    "\n",
    "def count_lines(filename):\n",
    "    count = 0\n",
    "    with open(filename) as fi:\n",
    "        for line in fi:\n",
    "            count += 1\n",
    "    return count"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'colorectal cancer is globally the third most incident malignancy 1 it is surgically treatable with improved long term outcomes if diagnosis is at an early stage 2 most developed countries including the uk operate colorectal screening programmes using faecal occult blood testing screen detected cancers benefit from early diagnosis and treatment with associated improved survival 2 faecal immunochemical tests fit have largely replaced the traditional guaiac based faecal occult blood tests due to the increased specificity of fit to complement the uk bowel cancer screening programme ukbcsp the 2017 dg30 nice guidance 3 recommended the use of fit for faecal haemoglobin hb detection in patients presenting to primary care with low risk abdominal symptoms the adoption of fit in primary care has been slow with notable variation in uptake and implementation across the uk 4 the oxford university hospitals trust ouh adopted fit prior to the dg30 guidance to comply with the 2015 ng12 nice guidance for suspected cancer 6 which recommended the use of faecal occult blood testing in symptomatic patients this coincided with a desire from the clinical laboratory to move away from the comparatively inaccurate guaiac based method fit was commissioned by oxfordshire clinical commissioning group occg as a direct access test for general practitioners in 2016 following a study by this group comparing the accuracy of the guaiac and fit methods in symptomatic primary care patients meeting ng12 criteria 5 despite gradual uptake of fit there remains a clear need to understand fit method characteristics 7 fit testing whether undertaken within a screening programme or applied to a symptomatic population is dependent on the analytical performance of the laboratory procedures used these analytical characteristics most crucially sampling impact on the way the results are most accurately reported 8 whilst there has been significant work undertaken in the context of screening 9 there is less data available in the symptomatic population this is important to explore as the characteristics of the population for a screening programme and the associated specimens may differ from patients presenting with symptoms for example the age range of the symptomatic population is broader and specimen characteristics including faecal consistency will likely differ in patients with changes in bowel habit and the information and support for taking a sample is less this may affect sample and sampling integrity the fit methods used to process samples from symptomatic patients are those developed for measuring faecal hb at higher concentrations in screening at present the cut off used by the ukbcsp is 120 Âµg g with future plans to reduce this to approximately 50 Âµg g to improve sensitivity the threshold recommended for use with symptomatic patients in dg30 is 10 Âµg g fit methods may not be optimised or fully characterised for detection at the lower concentrations now being considered important for application in dg30 7 while introduction of fit into primary care cancer pathways is limited at present given the large backlog of endoscopy created by the covid 19 pandemic it is almost certain the widespread use of low cut off fit testing will be introduced immediately the pandemic has passed to risk stratify all patient referred with possible cancer symptoms into groups for urgent and less urgent endoscopy understanding low cut off fit test performance in the context of symptomatic patients is therefore a priority in the present study we present our observations and assessment of fit method performance since introduction of the test into routine clinical practice for a symptomatic primary care population over a three year period we include estimates of analytical performance observations on longer term method performance sampling reproducibility in homogenised compared to non homogenised material and consistency is sequential specimens from the same patients a single laboratory undertaking centralised analysis of fit requests mainly from primary care for the 680 000 population of oxfordshire fit analysis was undertaken using the hm jackarc analyser kyowa medex co ltd tokyo japan ref guildford report which has a calibration range of 7 to 450 Âµg hb g faeces this method was introduced into service prior to nice guidance defining 10 Âµg g as the threshold defined for detection and we used 7 Âµg g the lowest calibrator value as the threshold for a positive result all specimens were requested by nhs primary care clinicians and collected into standard stool pots by patients a sampling approach taken due to concerns about sampling capability when undertaken by often elderly patients 5 clinician advice from ouh and occg included guidance on delivering the sample to the laboratory on the same day as collection as hb can degrade on storage 10 on arrival in the laboratory the stool specimens were sampled using the extel hemo auto mc designed for application with the hm jackarc by laboratory staff competency assessed in this technique the whole sampling procedure was undertaken in a fume cupboard by staff with standard laboratory protective equipment and involved a 30 second vortex mix which for most specimens was adequate for release of the faecal pellet collected into the dimples grooves of the picker all specimens were visually inspected to ensure complete suspension of specimen and if it was noted any residual specimen adherence mixing continued until all material was removed the fit method limit of blank lob limit of detection lod and limit of quantification loq were estimated as recently recommended 7 as it was not possible to obtain a genuine faecal sample assured as having no hb analyte present sample blank readings were utilised to estimate lob lod was calculated using the lob 1 645 sd of samples with very low concentrations considered to be those giving fit results those between 0 5Âµg g an 1 0 Âµg g loq was estimated from an imprecision profile constructed from faecal samples analysed in duplicate across a wide fit concentration range of naturally hb positive material and was taken as the concentration at which the percent coefficient of variation cv was 10 a 10 cv was selected as this has been recommended as an appropriate imprecision threshold for fit method loq evaluation 7 an additional within batch imprecision estimate was made using a dilution of the supplier s internal quality control iqc material this was prepared to give a concentration near to the anticipated lod to assess whether the behaviour of iqc material in the assay system with respect to imprecision was similar to true samples the on instrument intermediate imprecision not including sampling into the collection device was estimated from two level iqc materials provided by the manufacturer over several months to include the effects of different analysts reagent lot numbers and calibration material lot numbers it is unclear whether hb is evenly distributed in faeces therefore we investigated the hb concentration before and after sample homogenization randomly selected faecal specimens arriving in standard stool containers were sampled in triplicate homogenized and then resampled a further three times this resulted in six estimates of the result 3 nonhomogenised replicates and 3 homogenised replicates with the imprecision presented as the cv of each set of three replicates individual stool specimen replicates were sampled by the same member of staff but overall three different members of laboratory staff were involved in this part of the study fit analysis was undertaken as a batch once or twice per week to verify the stability of hb in the buffered collection device a selection of specimens were analysed at three time points at time zero at 6 days and at 21 days prior to january 2016 the standard practice with the guaiac method was to collect serial samples despite revised guidance recommending only a single specimen was required for fit many primary care clinicians continued to request more than one this may have been due to clinicians being unaware of revised guidance or because there remained doubt about test credibility and the additional reassurance multiple specimens might provide 11 12 sequential samples were delivered to the lab on the same day allowing assessment of delayed analysis on fit concentration these sequential specimens represented maximum within patient error as hb concentrations reflected total variability between day biological variation analyte instability sampling error and immunoassay method imprecision differences between the imprecision of sampling between non homogenised and homogenised sampling were assessed using a wilcoxon rank test with continuity correction the null hypothesis being that the distribution of x y cv in homogenised group cv in non homogenised group is symmetric about 0 for patients with multiple specimens and discordant results that is at least one result greater than and at least one result less than 10 ug g we examined whether there was an association between the likelihood of a positive result and delay in processing the specimen in the laboratory of 48 or 72 hours in this group we also examined whether there was an association between a positive result and the sequence number of the specimen both were assessed using the chi squared test limit of the blank lob the mean fit result of the sample blank run as part of the method iqc over 100 consecutive batches over 11 months and three different reagent batches was 0 20 Âµg g sd 0 16 Âµg g from which a derived lob was 0 47 Âµg g limit of detection lod as the method only reports to one decimal place we considered this to be 0 5 Âµg g for practical purposes 51 samples with apparent concentrations between 0 5 and 1 0 Âµg g measured in duplicate were considered very low concentrations and suitable for calculation of an sd for the lod calculation the derived sd was 0 35 Âµg g from which an lod of 1 1 Âµg g was derived from lob 1 645sd limit of quantification loq loq was calculated from a precision profile figure 1 constructed using data from 132 paired within batch replicates analysed in seven different batches over a one week period three samples with concentrations above the highest calibrant concentration 450 Âµg g were excluded from the analysis the imprecision was 13 at approximately 7 Âµg g the lowest calibrator value 11 5 at 10 Âµg g the nice threshold for detection and 10 at 15 Âµg g the 10 imprecision threshold has been recommended as the loq for fit 7 and therefore 15 Âµg g was considered to be this assay characteristic method precision and reproducibility the within batch imprecision characteristics of a highly diluted iqc material were mean 1 5 Âµg g sd 0 5 Âµg g cv 33 9 estimation of on instrument method precision utilised quality control data collected over 12 months during which time four different lot numbers were used table 1 method reproducibility expressed as cv for the low iqc material with a concentration of around 26 Âµg g ranged from 6 6 to 8 7 and for the high iqc material concentration around 90 Âµg g ranged from 4 5 to 6 8 effects of sample homogenisation 31 94 of 33 non homogenised samples were consistent with respect to their categorisation as above or below the 10 Âµg g figure 2 27 had a mean result below 10 Âµg g negative and 6 had mean results above positive four had all three replicates above the loq and imprecision estimates were calculated median cv of 27 8 range 20 5 to 48 6 two further samples had two replicates below the loq which prevented accurate calculation of imprecision but had discrepant result relative to the 10 Âµg g nice threshold one specimen had replicates of 1 1 1 1 34 5 Âµg g two below the lod and one above the loq clinical details of this patient were pr bleeding and endoscopy showed an 8mm polyp in the sigmoid histology revealed the polyp to be non dysplastic and hyper plastic and this patient was discharged the second of these discrepant results had replicates 1 1 1 1 14 7 Âµg g two below the lod and one between the lod and loq and positive relative to the nice threshold of 10 Âµg g this patient was being followed up by an established vague symptoms pathway 13 and whilst nothing abnormal was detected after 9 months of followup at the time of patient assessment they were noted to be taking nsaid in the homogenised specimen group all 33 100 specimens were consistent with respect to their categorisation as above or below the 10 Âµg g figure 2 28 had a mean result below 10 Âµg g negative and 5 had mean results above positive categorisation as positive or negative results was consistent in all replicates relative to a threshold of 10 Âµg g four of the specimens in the homogenised set had all three replicates above the loq and imprecision estimates were calculated median cv of 10 2 range 7 0 to 13 5 this was lower but not statistically significant p 0 10 than the imprecision observed in the non homogenised samples 15 specimens selected as their initial result was above the lod 8 below the loq and 7 above were repeat tested all 15 specimens had consistent categorisation when repeat tested at 6 days compared to the initial result positive or negative relative to the 10 Âµg g threshold at 21 days all but one specimen were consistently categorised however however one had fallen from 25 to 6 Âµg g for the 7 specimens above the loq there was no statistical difference at 6 p 0 95 or 21 days p 0 93 within patient serial sampling 524 patients sent more than one sample within a 3 month period 302 returned 2 specimens 222 returned 3 2 returned 4 and 1 returned 5 of the 225 patients with 3 or more specimens 197 87 6 had concordant results for all specimens 188 negative 9 positive 28 12 4 had discordant results that is at least one value more and at least one value less than 10 Âµg g there was no obvious trend on visual review of plots of individual patient s fit values over time figure 3 and no association between the likelihood of a positive result based on either a delay of more than 2 days in the specimen reaching the laboratory p 0 84 chi squared test there was no association between the likelihood of a positive results and the number in the sequence of a particular patient s specimens p 0 21 chi squared test all rights reserved no reuse allowed without permission the author funder who has granted medrxiv a license to display the preprint in perpetuity the copyright holder for this preprint which was not peer reviewed is https doi org 10 1101 2020 04 15 20066191 doi medrxiv preprint current proposals for fit testing at low concentrations are dependent on accurate lod and loq estimates 7 in this evaluation we have derived these detection characteristics for a commonly used fit method the lob 0 5 Âµg g lod 1 1 Âµg g are considerably lower than the hb concentration of 10 Âµg g designated by nice as the threshold for further patient investigation the loq if defined as the hb concentration where imprecision is 10 cv 7 was observed to be 15 0 Âµg g these detection capability estimates would suggest that at 10 Âµg g the method is sensitive enough to be confident that hb is present but not about the absolute value below 15 Âµg g the immunoassay method imprecision is good when assessed using non clinical iqc material being consistently 9 even over an 8 month period a diluted iqc material at a concentration 1 5 Âµg g near to the lod 1 1 Âµg g showed an imprecision of 33 9 higher than the precision profile line of fit derived with clinical material but reflective of highly variable imprecision at these lower concentrations the concentration of hb present in the two iqc materials provided by the manufacturer at around 25 Âµg g and 95 Âµg g these were presumably selected to be clinically appropriate for use in screening programmes where decisions thresholds are higher when the clinical interest shifts to lower concentrations we would suggest a strategy of using iqc materials with values below and above 10 Âµg g imprecision rises when faecal sampling is taken into consideration average imprecision in homogenised specimens was 10 2 and in non homogenised specimens was 27 8 with upper estimates of 13 5 and 48 6 respectively additionally inconsistent classification was observed with respect to the 10 Âµg g threshold in two further samples not assessed for imprecision due to some or all of the values being below the loq this would suggest that hb is not uniformly distributed in the faecal specimen a finding consistent with other faecal constituents for example a 1 h nmr spectroscopy study of faeces 14 highlighted both heterogeneity and instability of a range of metabolic constituents also whilst there is limited evidence to support the practice it is common for specimens for faecal elastase and calprotectin to include a homogenisation step prior to analysis future work must further establish the potential benefit of homogenisation in relation to fit we note that in two specimens one of the three non homogenised results was positive compared with none in the homogenised specimens this might lead to speculation that homogenisation leads to false negative results by reducing the amount of hb to the average in the stool specimen neither of these patients have had a diagnosis of colorectal cancer more than 1 year after the specimens were first obtained the stability of the hb in the sampling device buffer appears reasonable at 6 days and for most specimens up to 21 days however one specimen showed notable deterioration at 21 days from above to below the current 10 Âµg g threshold in a dg30 service setting a minimum weekly fit batch analysis would minimise the potential of this degradation to misclassify results our serial sample data provides some insights into sample variability and enables some comment on sample stability it is well established that hb degrades in faecal material 10 and this presents a potential for false negatives as the concentration may fall beneath that considered positive or detected in the serial samples from patients we do not know the magnitude of degradation in individual specimens but we found no statistical difference with respect to categorical positive or negative results with this time series of specimens our analysis is limited in that the intervals between specimens are variable it is possible that the variability between days a component of biological variability and from sampling a component of analytical variability may be larger than the concentration changes associated with degradation if sample degradation were the dominant variable it would be expected that the time series analysis would be significant comparisons to previous literature in a previous study 15 of the diagnostic accuracy of one vs two specimens it was concluded that two specimens offered no advantage over one specimen sequential results of 400 Âµg g were sometimes followed by 10 Âµg g from the same patient taken a median of 6 days apart it is not certain whether these differences are due to biological variation inconsistent bleeding or sampling imprecision a study 16 of the utility of two sample strategies for colorectal cancer in a symptomatic population it was noted that there was 39 2 nonconcordance between the first fit result and the maximum fit result it would therefore appear that whatever strategy is used bleeding from a lesion may be inconsistent and any single sample will on average detect with the same degree of diagnostic accuracy of note previous studies of fit in symptomatic patients have noted a small number of false negatives 4 in a recent study 17 using fit in a symptomatic population in a 2 week wait pathway 12 5 of colorectal cancers were missed using a single patient collected specimen using a threshold of 10 Âµg g the method utilised was different from the one utilised in our laboratory study the oc sensortm eiken chemical company tokyo japan and had a reported detection limit of 4 Âµg g as with other studies it is not certain whether these false negative results were due to variable patterns of bleeding poor sampling by the patient or another cause the study also noted that the majority of fit results from patients with colorectal cancer had very high values 150 Âµg g which would suggest these cases should be referred with higher priority there is on going debate as to how fit results should be reported from those that consider the result as detected not detected through to quantitative reporting of numeric values 4 7 given the high imprecision of analysis when sampling is taken into consideration we would caution against quantitative reporting of faecal hb due to the high uncertainty of measurement further audit of larger datasets derived from symptomatic patients in routine clinical practice are required to provide evidence of the association between hb concentration and clinical outcomes and to further inform qualitative reporting strategies the binary nature of reporting that we suggest may make for more straightforward pathways for the triage of patients who have been deferred due to the covid 19 pandemic and require re prioritisation into urgent and less urgent groups the quality assurance of faecal tests including fit presents unique analytical challenges compared to most clinical specimens used for diagnostic testing such as whole blood serum plasma or urine this is demonstrable in the lack of verification data in native faecal material available from manufacturers supplying sampling devices existing data depend upon synthetic material it also presents challenges to laboratories with respect to both iqc and eqa in our laboratory we have commenced analysis of samples collected directly by patients into the collection device and noted that despite collection guidance leaflets a youtube video and targeted training of staff advising patients many samples arrive with faecal material clearly on the outside of the container and no specimen apparent in the buffer anecdotally we have noted in many of the samples where sample integrity is questionable from visual inspection of the collection device the patient is over 75 years of age collection of samples in patients where there has been a change in bowel habit represents specific additional challenges and stool water content can vary between 60 and 82 18 the dilutional effect of this will impact on quantitative measures and the lack of form will affect entrapment of the specimen itself in the manufacturer s collection device we have noted a minority of specimens where considerably more than a 30 second vortex is required to suspend the faecal matter into the buffer protocols for how to manage these challenging specimens observed in the symptomatic population are lacking furthermore there is considerable concern about risk of faecal oral or aerosol transmission of covid 19 19 and the risks of sample manipulation and the need for a period of very high throughput need to be balanced against analytic accuracy the evaluation of collection device robustness is particularly important to continue one detailed study has reported that collection devices can be used effectively even when not used as recommended by the manufacturers 20 whilst it demonstrated significant differences in the absolute fit concentration the effect on diagnostic performance was small as with our study the testing involved specimen homogenisation and sampling in a laboratory to reduce variability in summary we have found that a commonly used fit method shows good consistency of categorising hb results against the dg30 threshold and therefore appears suitable for this clinical application however there appear numerous sources of variability that require further investigation and optimisation it is therefore important that all fit studies identify the relative contributions of biological variation sampling technique hb stability and method performance to false negative results as these are consistently observed in the literature the high imprecision precludes reporting of quantitative fit concentrations and on current analytical performance we consider a categorical approach for example negative or positive relative to recommendation of a 2 week wait referral is more appropriate potentially simplifying pathways to reprioritise patients with lower gi cancer symptoms who have had definitive diagnosis delayed by the covid 19 pandemic all rights reserved no reuse allowed without permission the author funder who has granted medrxiv a license to display the preprint in perpetuity the copyright holder for this preprint which was not peer reviewed is https doi org 10 1101 2020 04 15 20066191 doi medrxiv preprint figure 2 legend specimen preparation effects 33 specimens analysed in triplicate prior to and after homogenisation results show the observed concentration range lowest to highest with each specimen presented sequentially with homogenised circles and non homogenised triangles results adjacent the author funder who has granted medrxiv a license to display the preprint in perpetuity the copyright holder for this preprint which was not peer reviewed is the author funder who has granted medrxiv a license to display the preprint in perpetuity the copyright holder for this preprint which was not peer reviewed is table 1 imprecision estimates over a 12 month period all rights reserved no reuse allowed without permission the author funder who has granted medrxiv a license to display the preprint in perpetuity the copyright holder for this preprint which was not peer reviewed is https doi org 10 1101 2020 04 15 20066191 doi medrxiv preprint'"
      ]
     },
     "execution_count": 66,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "normalize_text(df_covid[\"body_text\"][0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [],
   "source": [
    "PRC_DATA_FPATH = \"../data/pretrain/proc_dataset.txt\" #@param {type: \"string\"}\n",
    "with open(PRC_DATA_FPATH, \"w\",encoding=\"utf-8\") as fo:\n",
    "    for l in df_covid[\"body_text\"]:\n",
    "        fo.write(normalize_text(l)+\"\\n\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### saving sentence per line "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "metadata": {},
   "outputs": [
    {
     "ename": "SyntaxError",
     "evalue": "invalid syntax (<ipython-input-90-e4c15c597b3b>, line 8)",
     "output_type": "error",
     "traceback": [
      "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-90-e4c15c597b3b>\"\u001b[0;36m, line \u001b[0;32m8\u001b[0m\n\u001b[0;31m    fo.write(sentencise_text(normalize_text(l))+\"\\n\")\u001b[0m\n\u001b[0m     ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
     ]
    }
   ],
   "source": [
    "PRC_DATA_FPATH = \"../data/pretrain/proc_sentence_dataset.txt\" #@param {type: \"string\"}\n",
    "sentencised =[]\n",
    "\n",
    "with open(PRC_DATA_FPATH, \"w\",encoding=\"utf-8\") as fo:\n",
    "    for l in df_covid[\"body_text\"][:10]:\n",
    "        sentencised.append(sentencise_text(normalize_text(I))\n",
    "\n",
    "        fo.write(sentencise_text(normalize_text(l))+\"\\n\")\n",
    "        fo.writelines(\"%s\\n\" % place for place in places_list)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Colorectal cancer is globally the third most incident malignancy (1) . It is surgically treatable with improved long-term outcomes if diagnosis is at an early stage (2) . Most developed countries, including the UK, operate colorectal screening programmes using faecal occult blood testing. Screen detected cancers benefit from early diagnosis and treatment, with associated improved survival (2) . Faecal immunochemical tests (FIT) have largely replaced the traditional guaiac based faecal occult blood tests due to the increased specificity of FIT.\n",
      "To complement the UK Bowel Cancer Screening Programme (UKBCSP), the 2017 DG30 NICE guidance (3) recommended the use of FIT for faecal haemoglobin (Hb) detection in patients presenting to primary care with low risk abdominal symptoms. The adoption of FIT in primary care has been slow, with notable variation in uptake and implementation across the UK (4). The Oxford University Hospitals Trust (OUH) adopted FIT prior to the DG30 guidance to comply with the 2015 NG12 NICE guidance for suspected cancer (6) , which recommended the use of faecal occult blood testing in symptomatic patients. This coincided with a desire from the clinical laboratory to move away from the comparatively inaccurate guaiac based method. FIT was commissioned by Oxfordshire Clinical Commissioning Group (OCCG) as a direct access test for General Practitioners in 2016 following a study by this group comparing the accuracy of the guaiac and FIT methods in symptomatic primary care patients meeting NG12 criteria (5) .\n",
      "Despite gradual uptake of FIT, there remains a clear need to understand FIT method characteristics (7) . FIT testing, whether undertaken within a screening programme or applied to a symptomatic population, is dependent on the analytical performance of the laboratory procedures used. These analytical characteristics, most crucially sampling, impact on the way the results are most accurately reported (8) . Whilst there has been significant work undertaken in the context of screening (9) there is less data available in the symptomatic population. This is important to explore as the characteristics of the population for a screening programme, and the associated specimens, may differ from patients presenting with symptoms. For example, the age-range of the symptomatic population is broader and specimen characteristics, including faecal consistency, will likely differ in patients with changes in bowel habit, and the information and support for taking a sample is less. This may affect sample, and sampling integrity.\n",
      "The FIT methods used to process samples from symptomatic patients are those developed for measuring faecal Hb at higher concentrations in screening. At present, the cut-off used by the UKBCSP is >120 Âµg/g, with future plans to reduce this to approximately 50 Âµg/g to improve sensitivity. The threshold recommended for use with symptomatic patients in DG30 is 10 Âµg/g. FIT methods may not be optimised, or fully characterised, for detection at the lower concentrations now being considered important for application in DG30 (7) .\n",
      "While introduction of FIT into primary care cancer pathways is limited at present, given the large backlog of endoscopy created by the COVID-19 pandemic it is almost certain the widespread use of low cut off FIT testing will be introduced immediately the pandemic has passed to risk stratify all patient referred with possible cancer symptoms into groups for urgent and less urgent endoscopy. Understanding low cut off FIT test performance in the context of symptomatic patients is therefore a priority.\n",
      "In the present study we present our observations and assessment of FIT method performance since introduction of the test into routine clinical practice for a symptomatic primary care population over a three year period. We include estimates of analytical performance, observations on longer term method performance, sampling reproducibility in homogenised compared to non-homogenised material, and consistency is sequential specimens from the same patients.\n",
      "A single laboratory undertaking centralised analysis of FIT requests, mainly from primary care, for the 680,000 population of Oxfordshire.\n",
      "FIT analysis was undertaken using the HM-JACKarc analyser (Kyowa Medex Co., Ltd., Tokyo, Japan) (ref Guildford report) which has a calibration range of 7 to 450 Âµg Hb/g faeces. This method was introduced into service prior to NICE guidance defining 10 Âµg/g as the threshold defined for detection and we used 7 Âµg/g, the lowest calibrator value as the threshold for a positive result All specimens were requested by NHS primary care clinicians and collected into standard stool pots by patients, a sampling approach taken due to concerns about sampling capability when undertaken by (often elderly) patients (5) . Clinician advice from OUH and OCCG included guidance on delivering the sample to the laboratory on the same day as collection as Hb can degrade on storage (10) . On arrival in the laboratory the stool specimens were sampled using the Extel Hemo-Auto MC designed for application with the HM-JACKarc by laboratory staff competency assessed in this technique. The whole sampling procedure was undertaken in a fume cupboard by staff with standard laboratory protective equipment and involved a 30 second vortex mix which for most specimens was adequate for release of the faecal pellet collected into the dimples/grooves of the picker. All specimens were visually inspected to ensure complete suspension of specimen and if it was noted any residual specimen adherence mixing continued until all material was removed.\n",
      "The FIT method limit of blank (LOB), limit of detection (LOD) and limit of quantification (LOQ) were estimated as recently recommended (7). As it was not possible to obtain a genuine faecal sample, assured as having no Hb analyte present, sample blank readings were utilised to estimate LOB. LOD was calculated using the LOB + 1.645 (SD) of samples with \"very low concentrations\", considered to be those giving FIT results those between 0.5Âµg/g an 1.0 Âµg/g. LOQ was estimated from an imprecision profile constructed from faecal samples analysed in duplicate across a wide FIT concentration range of naturally Hb positive material and was taken as the concentration at which the percent coefficient of variation (%CV) was 10%. A 10%CV was selected as this has been recommended as an appropriate imprecision threshold for FIT method LOQ evaluation (7) . An additional within batch imprecision estimate was made using a dilution of the supplier's internal quality control (IQC) material. This was prepared to give a concentration near to the anticipated LOD to assess whether the behaviour of IQC material in the assay system with respect to imprecision was similar to true samples.\n",
      "The on-instrument intermediate imprecision (not including sampling into the collection device) was estimated from two level IQC materials provided by the manufacturer over several months to include the effects of different analysts, reagent lot numbers and calibration material lot numbers.\n",
      "It is unclear whether Hb is evenly distributed in faeces therefore we investigated the Hb concentration before and after sample homogenization. Randomly selected faecal specimens arriving in standard stool containers were sampled in triplicate, homogenized and then resampled a further three times. This resulted in six estimates of the result (3 nonhomogenised replicates and 3 homogenised replicates) with the imprecision presented as the %CV of each set of three replicates. Individual stool specimen replicates were sampled by the same member of staff but overall three different members of laboratory staff were involved in this part of the study.\n",
      "FIT analysis was undertaken as a batch once or twice per week. To verify the stability of Hb in the buffered collection device a selection of specimens were analysed at three time points (at time zero, at 6 days and at 21 days).\n",
      "Prior to January 2016 the standard practice with the guaiac method was to collect serial samples. Despite revised guidance, recommending only a single specimen was required for FIT, many primary care clinicians continued to request more than one. This may have been due to clinicians being unaware of revised guidance or because there remained doubt about test credibility and the additional reassurance multiple specimens might provide (11, 12) . Sequential samples were delivered to the lab on the same day allowing assessment of delayed analysis on FIT concentration. These sequential specimens represented maximum within patient error as Hb concentrations reflected total variability (between day biological variation, analyte instability, sampling error and immunoassay method imprecision).\n",
      "Differences between the imprecision of sampling between non-homogenised and homogenised sampling were assessed using a Wilcoxon rank test with continuity correction, the null hypothesis being that the distribution of x -y (%cv in homogenised group -%cv in non-homogenised group) is symmetric about 0.\n",
      "For patients with multiple specimens and discordant results (that is, at least one result greater than, and at least one result less than 10 ug/g), we examined whether there was an association between the likelihood of a positive result and delay in processing the specimen in the laboratory of 48 or 72 hours. In this group, we also examined whether there was an association between a positive result and the sequence number of the specimen. Both were assessed using the Chi-squared test.\n",
      "Limit of the blank (LOB) -The mean FIT result of the sample blank run as part of the method IQC over 100 consecutive batches over 11 months and three different reagent batches was 0.20 Âµg/g, SD 0.16 Âµg/g, from which a derived LOB was 0.47 Âµg/g.\n",
      "Limit of detection (LOD) -As the method only reports to one decimal place we considered this to be 0.5 Âµg/g for practical purposes. 51 samples with apparent concentrations between 0.5 and 1.0 Âµg/g measured in duplicate were considered \"very low concentrations\" and suitable for calculation of an SD for the LOD calculation. The derived SD was 0.35 Âµg/g from which an LOD of 1.1 Âµg/g was derived (from LOB + 1.645SD).\n",
      "Limit of quantification (LOQ) -LOQ was calculated from a precision profile ( Figure 1 ) constructed using data from 132 paired, within-batch replicates, analysed in seven different batches over a one week period. Three samples with concentrations above the highest calibrant concentration, 450 Âµg/g, were excluded from the analysis. The imprecision was 13% at approximately 7 Âµg/g (the lowest calibrator value), 11.5% at 10 Âµg/g (The NICE threshold for detection) and 10% at 15 Âµg/g. The 10% imprecision threshold has been recommended as the LOQ for FIT (7) and therefore 15 Âµg/g was considered to be this assay characteristic.\n",
      "Method precision and reproducibility -The within-batch imprecision characteristics of a highly diluted IQC material were mean 1.5 Âµg/g, SD 0.5 Âµg/g, %CV 33.9 %. Estimation of on-instrument method precision utilised quality control data collected over 12 months during which time four different lot numbers were used (table 1) .\n",
      "Method reproducibility, expressed as %CV, for the low IQC material with a concentration of around 26 Âµg/g ranged from 6.6% to 8.7% and for the high IQC material, concentration around 90 Âµg/g ranged from 4.5% to 6.8%.\n",
      "Effects of sample homogenisation 31 (94%) of 33 non-homogenised samples were consistent with respect to their categorisation as above or below the 10 Âµg/g ( figure 2 ). 27 had a mean result below 10 Âµg/g (negative) and 6 had mean results above (positive). Four had all three replicates above the LOQ and imprecision estimates were calculated: median %CV of 27.8%, range 20.5% to 48.6%.\n",
      "Two further samples had two replicates below the LOQ which prevented accurate calculation of imprecision but had discrepant result relative to the 10 Âµg/g NICE threshold. One specimen had replicates of <1.1, <1.1, 34.5 Âµg/g (two below the LOD and one above the LOQ). Clinical details of this patient were PR bleeding and endoscopy showed an 8mm polyp in the sigmoid. Histology revealed the polyp to be non-dysplastic and hyper-plastic and this patient was discharged. The second of these discrepant results had replicates <1.1, <1.1, 14.7 Âµg/g (two below the LOD and one between the LOD and LOQ and positive relative to the NICE threshold of 10 Âµg/g). This patient was being followed up by an established vague symptoms pathway (13) and whilst nothing abnormal was detected after 9 months of followup, at the time of patient assessment they were noted to be taking NSAID.\n",
      "In the homogenised specimen group, all 33 (100%) specimens were consistent with respect to their categorisation as above or below the 10 Âµg/g ( figure 2 ). 28 had a mean result below 10 Âµg/g (negative) and 5 had mean results above (positive). Categorisation as positive or negative results was consistent in all replicates relative to a threshold of 10 Âµg/g. Four of the specimens in the homogenised set had all three replicates above the LOQ and imprecision estimates were calculated: median %CV of 10.2%, range 7.0 to 13.5%. This was lower but not statistically significant (p = 0.10) than the imprecision observed in the non-homogenised samples.\n",
      "15 specimens selected as their initial result was above the LOD, (8 below the LOQ and 7 above) were repeat tested. All 15 specimens had consistent categorisation when repeat tested at 6 days compared to the initial result (positive or negative relative to the 10 Âµg/g threshold).\n",
      "At 21 days all but one specimen were consistently categorised however, however one had fallen from 25 to 6 Âµg/g. For the 7 specimens above the LOQ there was no statistical difference at 6 (p=0.95) or 21 days (p=0.93).\n",
      "Within patient serial sampling -524 patients sent more than one sample within a 3 month period: 302 returned 2 specimens; 222 returned 3; 2 returned 4; and 1 returned 5. Of the 225 patients with 3 or more specimens, 197 (87.6%) had concordant results for all specimens (188 negative, 9 positive). 28 (12.4%) had discordant results, that is, at least one value more, and at least one value less, than 10 Âµg/g.\n",
      "There was no obvious trend on visual review of plots of individual patient's FIT values over time (figure 3), and no association between the likelihood of a positive result based on either a delay of more than 2 days in the specimen reaching the laboratory (p = 0.84, Chi-squared test). There was no association between the likelihood of a positive results and the number in the sequence of a particular patient's specimens (p = 0.21, Chi-squared test), All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.04.15.20066191 doi: medRxiv preprint\n",
      "Current proposals for FIT testing at low concentrations are dependent on accurate LOD and LOQ estimates (7) . In this evaluation we have derived these detection characteristics for a commonly used FIT method. The LOB (0.5 Âµg/g), LOD (1.1 Âµg/g) are considerably lower than the Hb concentration of 10 Âµg/g designated by NICE as the threshold for further patient investigation. The LOQ, if defined as the Hb concentration where imprecision is 10%CV (7) was observed to be 15.0 Âµg/g. These detection capability estimates would suggest that at 10 Âµg/g the method is sensitive enough to be confident that Hb is present but not about the absolute value below 15 Âµg/g.\n",
      "The immunoassay method imprecision is good when assessed using non-clinical IQC material, being consistently <9%, even over an 8 month period. A diluted IQC material at a concentration (1.5 Âµg/g) near to the LOD (1.1 Âµg/g) showed an imprecision of 33.9%, higher than the precision profile line of fit derived with clinical material but reflective of highly variable imprecision at these lower concentrations. The concentration of Hb present in the two IQC materials provided by the manufacturer, at around 25 Âµg/g and 95 Âµg/g. These were presumably selected to be clinically appropriate for use in screening programmes where decisions thresholds are higher. When the clinical interest shifts to lower concentrations, we would suggest a strategy of using IQC materials with values below and above 10 Âµg/g. Imprecision rises when faecal sampling is taken into consideration: average imprecision in homogenised specimens was 10.2% and in non-homogenised specimens was 27.8%, with upper estimates of 13.5% and 48.6%, respectively. Additionally, inconsistent classification was observed with respect to the 10 Âµg/g threshold in two further samples not assessed for imprecision due to some or all of the values being below the LOQ. This would suggest that Hb is not uniformly distributed in the faecal specimen, a finding consistent with other faecal constituents. For example, a 1 H NMR spectroscopy study of faeces (14) highlighted both heterogeneity and instability of a range of metabolic constituents. Also, whilst there is limited evidence to support the practice it is common for specimens for faecal elastase and calprotectin to include a homogenisation step prior to analysis. Future work must further establish the potential benefit of homogenisation in relation to FIT. We note that, in two specimens, one of the three non-homogenised results was positive compared with none in the homogenised specimens. This might lead to speculation that homogenisation leads to false negative results by reducing the amount of Hb to the average in the stool specimen. Neither of these patients have had a diagnosis of colorectal cancer, more than 1 year after the specimens were first obtained.\n",
      "The stability of the Hb in the sampling device buffer appears reasonable at 6 days and for most specimens up to 21 days. However, one specimen showed notable deterioration at 21 days from above to below the current 10 Âµg/g threshold. In a DG30 service setting a minimum weekly FIT batch analysis would minimise the potential of this degradation to misclassify results.\n",
      "Our serial sample data provides some insights into sample variability and enables some comment on sample stability. It is well established that Hb degrades in faecal material (10) and this presents a potential for false negatives as the concentration may fall beneath that considered positive or detected. In the serial samples from patients we do not know the magnitude of degradation in individual specimens but we found no statistical difference with respect to categorical positive or negative results with this time series of specimens. Our analysis is limited in that the intervals between specimens are variable. It is possible that the variability between days (a component of biological variability) and from sampling (a component of analytical variability) may be larger than the concentration changes associated with degradation: if sample degradation were the dominant variable it would be expected that the time series analysis would be significant.\n",
      "Comparisons to previous literature.\n",
      "In a previous study (15) of the diagnostic accuracy of one vs two specimens it was concluded that two specimens offered no advantage over one specimen. Sequential results of 400 Âµg/g were sometimes followed by <10 Âµg/g from the same patient taken a median of 6 days apart. It is not certain whether these differences are due to biological variation, inconsistent bleeding or sampling imprecision. A study (16) of the utility of two sample strategies for colorectal cancer in a symptomatic population it was noted that there was 39.2% nonconcordance between the first FIT result and the maximum FIT result. It would therefore appear that whatever strategy is used, bleeding from a lesion may be inconsistent and any single sample will, on average, detect with the same degree of diagnostic accuracy. Of note, previous studies of FIT in symptomatic patients have noted a small number of false negatives (4). In a recent study (17) using FIT in a symptomatic population in a 2 week wait pathway 12.5% of colorectal cancers were missed using a single patient collected specimen using a threshold of 10 Âµg/g. The method utilised was different from the one utilised in our laboratory study, the OC-SensorTM (Eiken Chemical Company, Tokyo, Japan) and had a reported detection limit of 4 Âµg/g. As with other studies it is not certain whether these false negative results were due to variable patterns of bleeding, poor sampling by the patient, or another cause. The study also noted that the majority of FIT results from patients with colorectal cancer had very high values (>150 Âµg/g) which would suggest these cases should be referred with higher priority.\n",
      "There is on-going debate as to how FIT results should be reported from those that consider the result as \"detected\"/\"not detected\" through to quantitative reporting of numeric values (4, 7) . Given the high imprecision of analysis when sampling is taken into consideration we would caution against quantitative reporting of faecal Hb due to the high uncertainty of measurement. Further audit of larger datasets derived from symptomatic patients in routine clinical practice are required to provide evidence of the association between Hb concentration and clinical outcomes and to further inform qualitative reporting strategies. The binary nature of reporting that we suggest may make for more straightforward pathways for the triage of patients who have been deferred due to the COVID-19 pandemic and require re-prioritisation into urgent and less urgent groups.\n",
      "The quality assurance of faecal tests, including FIT, presents unique analytical challenges compared to most clinical specimens used for diagnostic testing, such as whole blood, serum, plasma or urine. This is demonstrable in the lack of verification data in native faecal material available from manufacturers supplying sampling devices. Existing data, depend upon synthetic material. It also presents challenges to laboratories with respect to both IQC and EQA. In our laboratory, we have commenced analysis of samples collected directly by patients into the collection device and noted that, despite collection guidance leaflets, a YouTube video and targeted training of staff advising patients, many samples arrive with faecal material clearly on the outside of the container and no specimen apparent in the buffer. Anecdotally, we have noted in many of the samples where sample integrity is questionable from visual inspection of the collection device, the patient is over 75 years of age.\n",
      "Collection of samples in patients where there has been a change in bowel habit represents specific additional challenges and stool water content can vary between 60 and 82% (18) . The dilutional effect of this will impact on quantitative measures and the lack of form will affect entrapment of the specimen itself in the manufacturer's collection device. We have noted a minority of specimens where considerably more than a 30 second vortex is required to suspend the faecal matter into the buffer. Protocols for how to manage these challenging specimens observed in the symptomatic population are lacking. Furthermore there is considerable concern about risk of faecal-oral or aerosol transmission of COVID-19 (19) and the risks of sample manipulation and the need for a period of very high throughput need to be balanced against analytic accuracy.\n",
      "The evaluation of collection device robustness is particularly important to continue. One detailed study has reported that collection devices can be used effectively even when not used as recommended by the manufacturers (20) . Whilst it demonstrated significant differences in the absolute FIT concentration the effect on diagnostic performance was small. As with our study the testing involved specimen homogenisation and sampling in a laboratory to reduce variability.\n",
      "In summary we have found that a commonly used FIT method shows good consistency of categorising Hb results against the DG30 threshold, and therefore appears suitable for this clinical application. However, there appear numerous sources of variability that require further investigation and optimisation. It is therefore important that all FIT studies identify the relative contributions of biological variation, sampling technique, Hb stability and method performance to false negative results as these are consistently observed in the literature. The high imprecision precludes reporting of quantitative FIT concentrations and on current analytical performance we consider a categorical approach, for example negative or positive relative to recommendation of a 2 week wait referral is more appropriate, potentially simplifying pathways to reprioritise patients with lower GI cancer symptoms who have had definitive diagnosis delayed by the COVID-19 pandemic. All rights reserved. No reuse allowed without permission.\n",
      "the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.04.15.20066191 doi: medRxiv preprint Figure 2 legend: Specimen preparation effects. 33 specimens analysed in triplicate prior to and after homogenisation. Results show the observed concentration range (lowest to highest) with each specimen presented sequentially with homogenised (circles) and non-homogenised (triangles) results adjacent. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is Table 1 : Imprecision estimates over a 12 month period All rights reserved. No reuse allowed without permission.\n",
      "the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.04. 15.20066191 doi: medRxiv preprint \n",
      "The outbreak of infectious diseases has always been one of the most important health problems in the world. With the rapid spread of the new coronavirus (COVID- 19) , much attention has been paid to subjects like rate of an epidemic, transition ways, prevention methods, and remaining time of the virus in the environment [1] [2] [3] [4] [5] . Therefore, the behavior of the virus in relation to environmental factors is critical. The coronavirus is one of the epidemic diseases that many countries have experienced in the new type of it, COVID-19, and can become a serious challenge and affect the previous efforts on sustainable development [6] [7] [8] [9] .\n",
      "It has been found that there are some delays between the detection of confirmed cases and the actual infection date due to the laboratory test, the time of the announcement of confirmed cases in media, and the time for the patients showing initial symptoms for a total usually of 3 to 5 days [10] [11] [12] . The analysis of Kampf et al. (2020) demonstrated that COVID-19 could remain up to 9 days on different materials based on temperature and humidity [13] . Cascella et al. (2020) conducted research on the new coronavirus structure. Their analysis shows that the virus belongs to the family of single-stranded RNA viruses (+ ssRNA), which has a length of about 30 kb and an envelope with spear structures and they tested the sensitivity of the virus to UV and temperature [14] . investigated the correlations between environmental factors and the number of positive cases of COVID-19 by the use of artificial intelligence techniques. The results of this study put evidence on the role of urban and weather conditions on the epidemic rate [15] . Chen et al. (2020) developed a time-dependent mathematical model for the prediction of the total number of positive cases [16] . Lytle and Sagripanti (2005) studied the impact of UV rays, and their results determined that it can affect the structure of coronaviruses because UV rays could damage the nucleic acids of viral RNA that is vital for the survival of the virus [17] . developed an assessment method for investigating the impact of climate and urban factors in confirmed cases of COVID-19 [18] . Bukhari et al., (2020) , studied the epidemic of COVID-19 on tropical zones and their results show that the highest number of infection happened in temperature among 3 to 17 ÂºC [19] .\n",
      "The analysis of Grant and Giovannucci (2009) shows that by increasing the UV, the level of vitamin D in the body increases and makes the infection rates decrease. Therefore, in areas on lower latitudes and closer to the equator line, the epidemic on COVID-19 might be different [20] . In the study of Chan et al. (2011) , the effect of temperature and humidity on SARS coronavirus was analyzed, and they found that increasing temperature decreases the epidemic rate by faster disabling virus structure on contaminated surfaces [21] . Pirouz and Violini (2020) , analyzed the outbreak patterns of COVID-19 by analysis of 117 countries. They suggested two mechanisms of triggering the development of the infection, namely the normal spread of an infection from one parent case and the consequence of a unique phenomenon of accumulation of contagious people [22] . Analysis of the previous studies demonstrates the necessity of further research about the effect of environmental factors on the COVID-19. Since some of the previous researches have shown the effect of weather conditions, especially temperature, on coronavirus viruses, the main purpose of this study is to investigate the better correlation of daily mean air temperature on the epidemic rate of the COVID-19.\n",
      "In this study, we analysed the number of positive contagions of coronavirus COVID-19, from February 29 th to March 19 th . The selection of February 29 th is due to existence on this date of at least one positive case of COVID-19 in all case studies studied. The analysis of the weather data started from February15 th and ended on March 18 th because the impact of the weather parameter would occur in the upcoming days. Thus, the considered average daily temperature is with an interval of 1 to 14 days before the definitive cases of COVID-19.\n",
      "ï‚· In order to minimize the impact of other parameters on the results, such as the policy of each government, cultural differences between countries, health care systems, or the protocols in getting coronavirus test, comparative analysis have been done for different regions of the same country, without comparing with each other the results of different countries; ï‚· The analysis period for average daily temperature ranges from February15 th , 2020 to March18 th , 2020 (33 days); ï‚· The analysis period for confirmed cases of coronavirus COVID-19 is from February 29 th to March19 th ; ï‚· For what concerns the cumulative daily rate of confirmed cases, the total number of confirmed cases in a region was divided into the population of that region in order to take into account the different size of the population in each region.\n",
      "To carry out the analysis, the datasets of five countries and in each country, of three regions/provinces have been considered. The selected case studies are presented in Table 1 . and the location of the three regions shown in Figures 1. \n",
      "Mathematical models have been successfully used for the analysis of many hazards [32, 33] . In complex subjects, different Mathematical models can be used to solve the problem [34] [35] [36] . Statistical tests are an appropriate method for the investigation of possible correlations between different datasets [37] . Many statistical tests can be used for the analysis of significant differences between the datasets, among which F-test and Independent-Samples T-Test [38] .\n",
      "In order to investigate the hypothesis about a correlation between significant differences (Sig.) of average daily temperatures and average cumulative daily rate of confirmed cases in two regions, the following assumptions, and statistical tests have been done:\n",
      "ï‚· The variances of the two groups are equal: H0 ï‚· The variances of the two groups are not equal: H1\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04. 10.20059337 doi: medRxiv preprint When using F-test, if the significance level (sig) is less than 0.05, hypothesis H0 is rejected, and then hypothesis H1 is accepted, i.e., the variances of the datasets in the two selected regions are not equal. Conversely, if the significance level (sig.) is larger than 0.05, the hypothesis H1 is rejected, and the hypothesis H0 is accepted, i.e., the variances of the datasets in the two selected regions are equal.\n",
      "Through F-test, one determines the presence or absence of differences in the variances of the two regions. Through another statistical test, the Independent-Samples T-Test, one can determine whether the difference is significant, has been used. When using Independent-Samples T-Test, a significance level (sig.) lower (higher) than 0.05, means that the difference is (is not) significant.\n",
      "In our analysis, we used SPSS software. The critical factors in the SPSS Model set up are as follow:\n",
      "ï‚· The datasets in different regions of the same countries were analyzed with respect to the average daily air temperature during the 33-day period from 15-Feb to 18-Mar and the cumulative daily rate of confirmed cases of COVID-19 during the 19 days from 29-Feb to 19-Mar. ï‚· In all statistical tests, the confidence level of 95% is considered. ï‚· First, the F-test was used to test whether there is a significant equality of variances for each pair of case studies, and then, depending on the F-test result, by the T-Test the existence of a significant difference between the data of the two cases was investigated.\n",
      "The average daily temperature [39, 40] , and cumulative daily rate of confirmed cases [41] , are shown in Figures 2 to 6.\n",
      "a. Average daily temperature b. Cumulative daily rate of confirmed cases Correlations between the average temperatures and the cumulative daily rate of confirmed cases were found especially for towns with significant differences, with respect to the other two towns of the same country, in the average daily temperature. This was the case for Bandar Abas in Iran, Milan in Italy, Santa Cruz in Spain, and Los Angeles in the US. The average daily temperature and cumulative daily rate of confirmed cases are listed in Table 2 . As a general feature of Table 2 , we notice that ï‚· In 11 out of the possible fifteen comparisons, the average cumulative rate of confirmed cases is higher when the average daily temperature is lower, and in the four remaining cases the differences in either variable are small.\n",
      "A more specific analysis per country shows that ï‚· In Iran, the average daily temperature in Bandar Abbas was the highest (21.54 ÂºC), and the average cumulative rate of daily of confirmed cases was the lowest (0.000027). . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04. 10.20059337 doi: medRxiv preprint ï‚· In Italy, the average daily temperature in Milan was the lowest (7.99 ÂºC), and the average cumulative rate of daily of confirmed cases was the highest (0.000690). ï‚· In Germany, the average daily air temperatures of the selected towns have similar values, as well as the spread of the COVID-19 infection is similar. ï‚· In Spain, the average daily air temperature in Barcelona and Valencia are similar, and the spread of the infection is also similar. However, the comparison of Santa Cruz with higher average daily temperatures shows a slower spread of COVID-19. ï‚· In the United States, the average daily air temperature in both New York and Seattle are lower than Los Angeles, and the average cumulative rate of confirmed cases shows the spread of the virus in both is faster than in Los Angeles.\n",
      "These qualitative results reflect in the statistical analysis of the data by F-test and Independent-Samples T-Test presented in Table 3 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04. 10.20059337 doi: medRxiv preprint In Table 3 . t values or p-values can be used to check the existence of a significant correlation between the data for two cities.\n",
      "Let us discuss in detail one such comparison, that between Ramsar and Tehran. From the F-test, the F value is equal to 20.4. The criterion for variance equality in the F-test is that the F value must smaller than the critical F value. The statistical analysis has been done on a statistical confidence level of 95%, and therefore the error level is 0.05, and considering the degrees of freedom, the critical F value of the test that can be found through F distribution tables [42] , in this case, is equal to F = 1.57. Since the F value we found is 20.4, larger than the critical F value, the assumption of the equality of variances is rejected. It means that the average temperature variance of the Ramsar and Tehran are not equal.\n",
      "According to the t-test, the t value for these two cities is equal to -1.929. Based on an error level of 0.05 and by considering the degrees of freedom, the critical values of the test indicated in the T distribution tables [43] , is in this case equal to t = Â± 1.645. Since the t value is out of the critical boundary (-1.645 â‰¤ acceptable t â‰¤ +1.645), the assumption of equality in the averages is rejected, and the existence of difference in averages is confirmed. Thus from two tests, it is evident that there is a significant difference between the average temperature of the two cities of Ramsar and Tehran.\n",
      "An analogous result holds for the average cumulative rate of confirmed cases.\n",
      "The main results of Table 4 , with a confidence level of 95% are as follow:\n",
      "ï‚· In Iran, the city of Bandar Abbas is significantly different from Ramsar and Tehran in terms of both average daily temperature and average cumulative rate of confirmed cases. ï‚· In Italy, the city of Milan is significantly different from Rome and Naples in terms of both average daily temperature and average cumulative rate of confirmed cases. Instead, the two cities of Rome and Napoli are not significantly different in terms of these variables. ï‚· In Germany, there are no significant differences in average daily temperature in all three cities, and the average cumulative rate of confirmed cases is not significantly different. ï‚· In Spain, Barcelona and Valencia are not significantly different in terms of average daily temperature, and the average cumulative rate of confirmed cases is not significantly different. However, in Barcelona and Santa Cruz, the average daily temperature is significantly different, as well as the average cumulative rate of confirmed cases, and the same trend exists between Valencia and Santa Cruz. ï‚· In the United States, the two cities of New York and Seattle are not significantly different in terms of average daily temperature, and of the average cumulative rate of confirmed cases, the number of patients is not significantly different. Instead, the city of Los Angeles is significantly different in terms of average daily temperature with two cities of New York and Seattle, and the average cumulative rate of confirmed cases are different significantly.\n",
      "The results of our analysis of daily air temperature for three case studies from five countries showed the existence of correlations between the average temperatures and the epidemic rate, and this is especially evident when the differences in average daily temperature are larger. For example, in Iran and Italy, the regions with lower average daily temperature had a higher epidemic level of the new coronavirus (COVID- 19) , and the same trend exists in all other case studies.\n",
      "The statistical analysis showed that in all the case studies we considered, referring to five different countries, if there is a significant difference in daily mean temperature, there is also a significant difference in the average cumulative daily rate of confirmed cases, supporting the proposed research hypothesis about the correlation between air temperature and the epidemic rates of COVID-19. Due to the correlations among temperature and hours of sunshine for future research, it is recommended to study the effects of the sun's UV rays as well as the average hours of sunshine on the epidemic rates of COVID-19.\n",
      "This study analyzed the possible correlation of daily average air temperature with the epidemic rate of the coronavirus (COVID-19) using significant differences analysis between the datasets. The research hypothesis is that in regions with a significant difference in the average daily temperature, the average cumulative daily rate of confirmed cases would also is different significantly. Conversely, to the absence of a significant difference in daily temperature, would correspond no significant difference in the rate of confirmed cases. The hypothesis was investigated through statistical analysis. In order to minimize the impact on the results of factors like the policy of the government or cultural differences among countries (food, exercise, weight, etc.), three case studies within five countries, namely Iran, Italy, Germany, Spain, and United States were compared separately. This comparison using statistical analysis through F-test and Independent-Samples T-Test of the daily temperatures and daily rate of confirmed cases shows that there is a correlation between the average temperature and the epidemic rate, and this is especially evident when differences in average daily temperature are significantly larger, as it happens for Bandar Abbas in Iran, Milan in Italy, Santa Cruz in Spain, and Los Angeles in the US. Besides, the analysis of the average air temperatures shows that the epidemic rates of COVID-19 were higher in the case studies with a lower average temperature. Instead, when no significant differences exist in the average daily temperature of two cities in the same country, there is no significant difference in the average cumulative daily rate of confirmed cases. In conclusion, the results of this study support the research hypothesis and confirm the effectiveness of the proposed method for analysis of the epidemic rates.\n",
      ". CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04. 10 . CC-BY-NC-ND 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10.1101/2020.04. 10.20059337 doi: medRxiv preprint \n",
      "In the absence of a curative treatment, the highly active antiretroviral therapy (HAART) keeps the HIV-1 virus of AIDS patients under control. HAART combines drugs targeting different stages of viral replication including the integration step catalyzed by the integrase protein (IN) (1) . Integration of viral DNA into host genome involves two steps catalyzed by IN: (i) cleavage of a dinucleotide from each 3'-end of the viral DNA (3'processing), and (ii) insertion of this processed viral DNA into the host DNA (strand-transfer) (2) . Clinical IN strand transfer inhibitors (INSTIs) target the catalytic site of the enzyme to specifically inhibit the DNA joining reaction, however, as with all anti-AIDS treatments, the continued success of these drugs is persistently disrupted by resistance mutations (1, 2) . Although 3'-processing can be carried out by monomeric IN (3) , the assembly of IN functional multimers is imperative for the strand-transfer activity (4) (5) (6) (7) (8) , and for virus particle maturation and production (reviewed in (9, 10) ). In the continued quest to identify and develop new drugs, allosteric inhibitors that bind sites outside the catalytic core and disrupt IN multimerization are emerging with potent therapeutic potential (11) (12) (13) (14) .\n",
      "Tetramers of IN are formed by the reciprocal swapping of the three, N-terminal, catalytic core and Cterminal, canonical domains of IN. The two internal IN protomers, where catalytic binding of viral and host DNA takes place, make up the majority of the tetramer interface. The outer two protomers, and other surrounding units in higher order assemblies, provide supportive domains indispensible for the assembly of the tetrameric cores (4) (5) (6) (7) (8) . We recently showed that targeting the interface between the N-terminal domain (NTD) and catalytic core domain (CCD) of IN, using a specific antigen-binding fragment (Fab), inhibits IN tetramer formation and consequent enzymatic activity and virus particle production (9) . Disrupting the dimerization of the elementary dimeric blocks building IN tetramers has also been explored as a strategy to inhibit IN activity (15) .\n",
      "Hindering the assembly of IN functional multimers is only one side of the coin. Allosteric interference has also been shown to promote the formation of aberrant IN multimers and aggregates. The potential of allosteric IN inhibitors has been demonstrated through the thorough characterization of the \"LEDGF pocket\" formed at the dimer interface of IN and the development of LEDGIN (or ALLINI) inhibitors that bind to it (11) (Figure 1A) . Although less investigated, other IN pockets capable of allosteric inhibitor binding have also been identified ( Figure 1A) : binding of the \"Y3\" molecule to a pocket near the N-terminal end of CCD Î±-helix 4, designated Y3pocket, has been shown to inhibit 3'-processing and strand-transfer activities (16) ; the \"sucrose\" binding pocket found along the CCD dimer interface and flanked by two LEDGF pockets (17, 18) has recently been targeted by the natural product kuwanon-L, which inhibited IN activity in a pattern similar to LEDGINs (19) . Another fragment-binding pocket \"FBP\" has also been identified at the CCD dimer interface (20) . Although inhibitory profiles for this FBP pocket have not been shown, our previous work on IN from the feline virus showed that a single Phe187Lys mutation (Phe185 in HIV-1 IN) around this site inhibits dimer formation (3) . Fragment-based screening and structural studies have revealed the capability of all three pockets (Y3, FBP and LEDGF) to bind numerous small molecules (21) . Most recently, a new class of LEDGINs has been shown to distinctively bind to undefined interfaces of both CCD and CTD of IN (22) . Therefore, IN harbors several allosteric hotspots that can be explored for novel anti-multimerization intervention strategies.\n",
      "Based on the structure of Fab specific to IN CCD/NTD platform (9), we develop a peptide that, similar to Fab, inhibits IN multimerization and strand-transfer activity. Surprisingly, crystal structure and affinity experiments show that the peptide interferes with the CCD/CTD interfaces of IN. This structure, which features an overlooked SH3-docking platform crucial for IN multimerization, can now provide a template for the screening and development of novel anti-IN allosteric inhibitors.\n",
      "Within a functional IN tetramer, the NTD and CTD of one dimer wrap around an extended Î±-helix (Î±4) of CCD of a second dimer ( Figure 1B) , an interdomaindocking platform that is functionally imperative (9) . Previously, we showed that complementarity-defining regions 3 (CDR3) and 2 (CDR2) of an anti-IN Fab wrap around Î±4 of IN CCD (residues 153-168) and dock into a cleft rimmed by the CCD finger loop (residues 186-191) ( Figure  1C ). We derived a peptide (WSYFYDGSYSYYDYESY) mostly representing the CDR3 sequence (underlined) with the addition of Trp (for UV-Vis absorbance) and Ser-Tyr (representing CDR2 extension in wrapping around Î±4). Similar to parental Fab, the peptide inhibited IN strand-transfer activity but not 3'processing ( Figure 1D ), suggesting potential interference with IN functional multimerization.\n",
      "To evaluate the effects of the peptide on IN multimerization we used chemical crosslinking and found the peptide to indeed interfere with IN multimerization ( Figure 1E ). This underscores the allosteric potentials of the site(s) targeted by the peptide, and clarifies that the inhibition of IN multimerization and activity by the parental Fab (9) was specific to CDR3/CDR2 local allosteric interference and not an artifact of the bulkier size of whole Fab.\n",
      "To characterize the peptide binding site(s), we solved the 2.0 Ã… crystal structure of HIV-1 IN CCD in complex with the peptide (table-1). Except for the Cterminal pair of residues (Ser-Tyr), well-defined electron densities show binding of the peptide molecule to two sites, which we call site-1 and site-2 ( Figure 2A ). By examining available structures of various IN functional multimers (4) (5) (6) (7) (8) we underscore the conservation of these two CCD/CTD docking platforms ( Figure 2B ). Site-1 is at the tetrameic interface and is formed by a CTD from a flanking protomer docking at the carboxyl-terminus of CCD Î±4 of an inner protomer ( Figure 2B & 1B) . Forming site-2 is the docking of a CTD of another flanking protomer into the amino-terminus of the same Î±4 ( Figure  2B & 2C). The surface area buried upon peptide binding at site-2 is ~ 100 Ã… 2 larger than that at site-1 (744 as compared to 650 Ã… 2 ). Peptide binding at site-2 also involves a higher number of interacting residues at the interface (26 as compared to 17 at site-1). Side chains of two residues of the peptide (Trp1 and Tyr3 at site-1 and Ser10 and Tyr14 at site-2) participate in direct hydrogen bonds with the CCD (to side chains of Glu69 and Asp167 at site-1, and to carbonyl oxygen of Val79 and Ala80 at site-2). Additional hydrogen bonds were directly formed with the backbone of the peptide (one in site-1 and three in site-2), and water molecules mediated additional interactions at each site (two in site-1 and one in site-2) ( Figure 2D ). Interestingly, Ala80 carbonyl oxygen of CCD at site-2, which makes a hydrogen bond with the peptide, also makes a hydrogen bond (2.93 Ã…) with Lys266 of CTD in the HIV-1 IN functional multimers (discussed below and Figure 4B ). It has been shown that the positive charge at this Lys266 position is crucial for viral replication in which a Lys266Ala or Lys266Glu produced replication defective viruses (23) .\n",
      "Whereas the peptide mainly represented the CDR3sequence and was expected to also represent CDR3 binding pattern (to dock at the CCD/NTD interface between Î±4 and the finger loop, (Figure 1C )), the peptide surprisingly docked underneath Î±4 to mostly resemble the binding of CDR2 at site-1 ( Figure 2A ). Superimposing the CCD-peptide structure onto that of the HIV-1 intasome reveals potential steric interference with the CCD docking platform of CTD but not NTD at site-1 ( Figure 2E , top). Similarly, the symmetry related molecule of the peptide bound at site-2 reveals potential interference with the docking of another CTD ( Figure 2E , bottom).\n",
      "To demonstrate the ability of a fragment specific to site-2 to interfere with CCD/CTD docking and consequently inhibit IN multimerization, we evaluated the inhibitory effects of the Y3-molecule, which specifically binds at site-2 (16) . Indeed, using chemical crosslinking we show that the Y3-molecule attenuates IN . CC-BY 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/301721 doi: bioRxiv preprint multimerization in a dose-dependent manner ( Figure 2F ), corroborating on the potentials of a site-2 bound peptide to inhibit IN multimerizaion. Therefore, peptide binding at either of these sites, or both, would inhibit CTD interactions with CCD, and can rationalize the observed inhibition of IN multimerization ( Figure 1E ) and strand-transfer activity ( Figure 1D ). It is worth noting that since both docking sites involve not only the CTD but also the NTD linker segment (residues 50-58) ( Figure 2B ), peptide binding would also interfere with the vital docking of NTDs during IN multimerization.\n",
      "Competency of peptide-bound IN for 3'processing activity indicates that peptide binding did not restrict the catalytic loop flexibility (density map is missing for residues 140-147) nor induced inactivating conformational changes to the enzyme (root-men-square deviation [RMSD] 0.30 Ã… to apo, PDB code: 1BIS).\n",
      "The higher number of interactions and the larger surface area buried upon peptide binding to site-2 may highlight a binding privilege at this site. Analyses of crystal structures of IN truncation variants, which have been shown to dimerize (CCD and CCDCTD) or tetramerize (NTDCCD) in solution and crystals, show the clear exposure and accessibility of site-1 ( Figure 3A and B). Whereas site-2 is also exposed in the crystal structures of dimeric CCD and tetrameric NTDCCD ( Figure 3A ), analyses of CCDCTD crystal structures intriguingly show that CTD is constantly docked into site-2, albeit in various configurations distinct to that of the functional intasome ( Figure 3B , orange cartoon). If indeed the CTD preoccupies site-2 within the truncated CCDCTD variant, as seen in crystal structures, then we expect that, unlike CCD or NTDCCD variants, the CCDCTD construct would not bind the peptide. Assessing peptide-binding affinities to the various IN constructs show that the presence of NTD (within NTDCCD) only slightly (~ 1.5 folds) interferes with peptide binding ( Figure 3C ). CTD (within CCDCTD), on the other hand, almost completely abolishes peptide binding to more than 12 folds ( Figure  3C ). In agreement with the binding data, structural superimposition of the peptide shows that while NTD (of NTDCCD structure) may delicately interfere with peptide binding at site-1 ( Figure 3A ), CTD (of CCDCTD structures) would provide a more robust interference at site-2 ( Figure 3B ). It is worth noting that the peptide bound to full-length IN, which is preassembled into tetramers in solution (24) , with affinities (Kd 1700 Â± 88 nM) corresponding to the concentrations in the monomerization assay ( Figure 1E ) and ~ 4 folds weaker than NTDCCD (Kd 441 Â± 11 nM, Figure 3C ). Together, the NTD appears to crucially modulate CTD interactions with the CCD.\n",
      "The CTD preoccupation of site-2 in the twodomain CCDCTD construct may have significant implications on both the physiologic and the aberrant interactions mediated by the SH3-containing CTD domain. The incompetence of CCDCTD variant for 3'-processing activity (25, 26) indicates that the CCD/CTD configuration seen in the crystal structures of this construct ( Figure 3B ) is aberrant. Conceivably, interdomain interactions, especially those provided by the NTD and missing in CCDCTD, are required for correct CTD interactions and to maintain the catalytic site of the enzyme accessible for catalytic activities.\n",
      "To gain further insights into the interactions made by NTD and CTD, and their interference with the catalytic activity of CCD, we assessed 3'-processing activity of truncated domain variants of HIV-1 IN. These truncation variants are not active in strand-transfer activity (24) (25) (26) , a result that we also validated here (not shown). We found that whereas the CCD (residues 56-209) and NTDCCD (residues 1-209) variants of HIV-1 IN are as competent for the 3'-processing activity as the full-length enzyme, the CCDCTD (residues 56-288) is indeed defective ( Figure  4A ).\n",
      "The inhibitory effect of CTD, as part of the CCDCTD, was diminished upon separating the CTD segment (residues 210-288) and supplementing it to the CCD protein in the reaction mixture ( Figure 4A , CCD/CTD). Furthermore, the addition of CTD to a reaction containing the separate CCD and NTD (residues 1-55), which by itself ( Figure 4A , CCD/NTD) was found more than 3 folds superior to full-length or conjugate NTDCCD, resulted in a significant 33% inhibition ( Figure  4A , CCD/NTD/CTD). Therefore, while NTD positively interferes with the CCD in the 3'-processing activity, the CTD appears to have a negative effect. The contribution of the NTD appears necessary to support appropriate CTD interactions with the CCD. Structural analysis shows that the NTD linker segment (residues 50-58) contributes 55% (725 Ã… 2 ) of the total 1310 Ã… 2 buried CCD/CTD area at site-2 ( Figure 4B) . Interestingly, the carbonyl oxygen of Val54 of NTD linker forms a hydrogen bond (2.91 Ã…) with the side chain of CTD Arg228 (NH2) (Figure 4B ), mutating of which to Arg228Ala resulted in defective HIV-1 viruses (23) . The side chain of NTD Gln53 (NE2) forms a hydrogen bond (2.50 Ã…) with the carbonyl oxygen of CTD Asp229 (Figure 4B) , and also interacts with the side chain of Lys264 (3.6 Ã…). Both Lys264 and Lys266, which hydrogen bond to Ala80, have been shown to play an important role in the LEDGINs induced aberrant multimerization (27) . Indeed, the fundamental role of CTD . CC-BY 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/301721 doi: bioRxiv preprint in aberrant IN aggregation has previously been emphasized (10, 27, 28) . CTD interactions with the NTD linker (residues 44-53) at site-1, which provides the main docking template for CTD at this site, buries a total of 944 Ã… 2 ( Figure 4C) . Therefore, the NTD linker appears to play a crucial role in modulating the CCD/CTD docking platform and functional IN mutlimerization and is worth exploring in future studies.\n",
      "Multimerization of HIV-1 IN depends on the delicate compatibility of amino acids lining the interacting interfaces (9), providing an attractive targeting strategy. Interfering with the interacting domains can be exploited not only to promote aberrant multimerization but also to inhibit the assembly of functional multimers, an opportunity offered by the CCD/NTD (HTH-cleft) (9) and the CCD/CTD (SH3-cleft) docking platform reported here. The SH3-cleft (site-2) relates to a previously identified \"Y3\" pocket observed almost two decades ago in a crystal structure of retroviral IN CCD bound to the small molecule (Y3) ( Figure 1A) , which efficiently inhibits IN catalytic activity (16) . A recent fragment-based screening study has also captured several other small molecules bound near this site (21) (Figure 1A, CDQ) , however, the inhibitory profiles where not characterized.\n",
      "The inability of peptide to bind CCDCTD, which possesses an exposed site-1 but blocked site-2, indicates that site-2, which docks the CTD during multimerization, is a preferred binding site. CTD involvement in the multimerization of IN is well documented (reviewed in (23)), and the relative orientation of the CTD with regard to the CCD has previously been implicated in functional IN assembly (29) . Moreover, analysis of available CCD structures bound to small molecules at site-2 reveals a pocket that forms upon binding, which displaces the flexible N-terminus of Î±4, a displacement that would also interfere with and inhibit the binding of viral DNA and IN catalytic activity ( Figure 4D ). Therefore, we propose site-2 for future structure based design or screening of small molecules that can interfere with CCD/CTD interactions during IN multimerization, and can promote the formation of a binding pocket that would also inactivate the catalytic site by displacing Î±4 into an aberrant configuration.\n",
      "The fact that IN regulates not only viral integration but also the assembly and maturation of virus particles (9, 22) , accentuates the prominence of this target. Whereas the peptide used in this study may not, per se, provide a lead hit for this anti-IN strategy, our work underscores the overlooked SH3-docking platform of HIV-1 IN as a potential therapeutic target for future anti-IN allosteric inhibitors.\n",
      "All HIV-1 IN constructs were derived from the previously described variant (SF1: (SF1; C56S, W131D, F185K and C280S (24)) and subcloned into pET28b plasmid (Novagen) with a cleavable N-terminal His-tag. The various constructs contained the following residues: full-length 1-288, NTD 1-55, CCD 56-209, NTDCCD 1-209, CTD 210-288 and CCDCTD 56-288. To include the NTD linker, the NTD 1-55 used here was made longer than our previous construct (residues 1-51 of HFH chimera (9)). All constructs were confirmed by DNA sequencing.\n",
      "IN constructs were expressed and purified as previously described (24) . Lysis buffer contained 50 mM potassium phosphate (pH 7.5), 5 mM Imidazole, 5 mM Î²mercaptoethanol and 1M NaCl for the full-length protein or 0. \n",
      "Fluorescence based IN activity assays were performed as previously described (30) . Fab was used at a 1:1 molar ratio and the peptide at an excessive ratio of ~ 20 folds. and were processed with iMosflm (31) . The structure was solved by Phaser molecular replacement (32) using IN CCD (PDB code: 1BIS (33)) as a search model. Electron densities were fitted using Coot (34) and refined in the CCP4 suite (35) using REFMAC5 (36) . Structural superposition and figures were prepared using PyMOL Molecular Graphics System (SchrÃ¶dinger, LLC). Table-1 summarizes data collection and refinement statistics.\n",
      "Analyses of interactions were performed using COCOMAPS (37) and LigPlot + (38) . Lysine Crosslinking: Homobifunctional BS 3 (Pierce, 2.5 mM) was added to full-length IN (20 ÂµM) that was initially mixed with increasing concentrations of inhibitory peptide or Y3molecule (4-Acetamido-5-hydroxy-2,7-naphthalenedisulfonic acid, (Sigma, S981680)). Reactions were incubated for 30 minutes at room temperature, and quenched by adding Tris-HCl (pH 7.5) to a final concentration of 10 mM and analyzed on SDS-PAGE. Affinity Measurements: IN proteins were labeled with Monolith NT protein labeling kit BLUE-NHS (NanoTemper Technologies) and eluted in microscale thermophoresis (MST) buffer (50 mM Tris-HCl pH 7.0, 300 mM NaCl and 0.05% Tween-20). Labeled IN proteins were incubated with serial dilutions of inhibitor peptide, and MST measurements were carried out using the Monolith NT.115 in standard capillaries with 40% LED and 20% MST power.\n",
      "Atomic coordinates and structure factors were deposited in the Protein Data Bank with the accession number 6EX9. . CTD of 1EX4 is modeled from a symmetry mate. Site-1 (S1) of all CCDCTD structures is clearly exposed and therefore a peptide was not superposed at S1. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/301721 doi: bioRxiv preprint\n",
      "the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is . https://doi.org/10.1101/2020.03.28.20043059 doi: medRxiv preprint 3 except for one due to an unrelated bacterial infection) ( Table S1 in fecal samples, and the highest virus titers at the peak was significantly higher for 62 feces (10 5.8 copies/ml, mean 5623 copies/ml) than of respiratory samples (10 6.3 63 copies/ml, mean 2535 copies/ml) ( Figure 1A) . Notably, at 26 days after discharge, 64 case 3 was detected positive again in feces samples, but appears to be healthy and 65 negative for respiratory swabs. These results indicated that beside respiratory samples, 66 All rights reserved. No reuse allowed without permission. the author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is \n",
      "The pandemic of COVID-19 is taking a troll in health and finance sector of all most 2 every country. The Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV2) 3 originated from Wuhan a city in China is mutating while getting transmitted [3] . 4 Development of a potential drug or vaccine is a major challenge. The virus has RNA as 5 a genetic material and it gets mutated frequently. It is very important to understand 6 the evolutionary feature in the virus. Understanding the nature of mutation may help in 7 developing potential drug. Also, it may give direction to understand the next cycle of 8 mutation in the viral genome. Therefore, it is very important to understand the nature 9 of variation in different strains of the novel virus. 10 In this regard, few studies are conducted to understand the nature of variation in 11 this novel virus. Carmine and Federico [4] made an attempt to understand the 12 heterogeneity in 54 strains of this viral genome. Rahila et al. [16] studied impact of the 13 mutations in the spike glycoprotein function and stability in the novel virus.\n",
      "14 Changchuan [20] presented a method for effectively genotyping SARS-CoV-2 viruses 15 using complete genomes. 16 In this report, an analysis of 468 novel and early virus genome is presented. The 440 17 sequences are from different parts of the world including countries from Europe, USA, 18 Middle East, Asia and 28 sequences submitted from India with 11 sequenced in Iran. 19 The data was downloaded from GenBank, GISAID, and SRA. Data was collected for 20 this study last as of 9th April 2020, with 440 genome sequence of the novel and early 21 strains from the world along with 28 sequences from India. The MEGAX tool [11] has 22 been used to perform the analysis. It is based on Tamura-Nei model and 23 Neighbour-Joining method along with 100 bootstrap. This study adds up a novel 24 non-synonymous mutation 4809C>T (S1515F) in NSP3 gene only observed in strains 25 obtained from India. Various studies revealed that the NSP3 gene harbours many 26 mutations as this gene has been reported with extensive positive selection towards 27 evolution in Betacoronaviruses [18] , [7] . Common mutations among different strains 28 were also identified and reported. The sequence containing the mutation was used to 29 build protein model using SWISS-Model. Followed by the usage of the DynaMut tool to 30 study effect of mutation on protein stability. The results indicate the clade of variations 31 in the Indian SARS-CoV2. Also, potential region of mutations in near future along with 32 the effect of non-synonymous mutation at the protein level. This paper represents the 33 initial observations on submitted 28 sequences of Indian viral strains.\n",
      "Sequence collection: total 468 sequenced strains were selected, 440 from different parts 36 of the world including countries from Europe, USA, Middle East, Asia and 28 sequences 37 submitted from India with 11 sequenced in Iran. Sequences were downloaded from 38 GenBank, GISAID, and SRA project (PRJNA607948). Information of samples used in 39 this study are provided as supplementary material.\n",
      "SRA project sequences were aligned to human genome assembly (hg19) using 41 HiSat2 [10] , to remove the human contamination in viral sequences. Next the unaligned 42 region to hg19 was aligned to SARS-CoV2 ref sequence (NC 045512) using 43 SamTools [13] and HiSat2. Consensus fasta sequence of four viral strains of project were 44 obtained using BcfTools [13] . with Tamura-Nei model and Neighbour-Joining method along with 100 bootstrap steps 52 using the MEGAX.\n",
      "This section describes about the different types of mutations discovered in the analysis 55 along with an analysis of the mutation at protein level.\n",
      "The Common Mutations 57 Genome sequences of 440 viral strains submitted from different parts of the world were 58 collected from GenBank, GISAID and SRA project. In order to investigate the changes 59 in 28 viral strains submitted from India a sequence level and phylogenetic analysis was 60 performed. Multiple sequence alignment of these collected sequences was visualised to 61 collect the positions in Indian strains with mutations.\n",
      "There has been a report on SNP mutations in SARS-CoV2 genome which majorly 63 demonstrated mutations frequent in European strains [20] . These mutations were 64 analysed in 28 submitted sequences from India. A previous report from India observed 65 unique mutation in spike protein (A930V (24351C>T)), this mutation was identified in 66 one (Sample 2) of the early infected patients' viral sequence [16] . The most common 67 SNP's that were observed in this analysis includes the previously reported mutation in 68 leader sequence (241C>T), Coronaviruses includes subgenomic identical 5 Leader 69 sequence which plays role in viral replication process. This mutation was seen to replicase-transcriptase complex at a very early stage of infection [8] . Spike protein has 89 been a hotspot in CoV2 genome since the beginning due to the presence of 90 receptor-binding domain. Contact point to the host includes the domains from S1 91 subunit of this S-protein [19] . The mutation observed 23403A>G (D614G) lies in tail 92 region of this S1 subunit and before the cleavage site of another subunit called S2 of 93 S-protein. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.14.041301 doi: bioRxiv preprint Talking in terms of RNA replication of SARS-CoV ORF1a and ORF1b 95 (replicase/transcriptase) which includes NSP8 and NSP4 genes have been highlighted. The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.14.041301 doi: bioRxiv preprint SARS-CoV2 has been reported to have second highest identity (82.30%) with 146 SARS-COV. To identify the possible role and relevance of the region observing the 147 mutation, sequence alignment of NSP3 protein from SARS-CoV and SARS-CoV2 was 148 performed. NSP3 gene along with NSP4 are the key players for membrane 149 rearrangement in form of Double Membrane Vesicles (DMV) upon infection [2] . Report 150 on role of mutations in important glycosylation sites in NSP4 gene in SARS-Cov have 151 been observed and were indicative of localisation problems with other NSP protein like 152 NSP3 [5] . As per this there will be a requirement to observe the site of novel change for 153 its impact on NSP3 gene interactions, which may even serve as a potential candidate for 154 therapeutics, depending upon the mutational effect shown in NSP4 [5] . The other 155 predictive outcome can come as benefit to virus in terms of interaction. recognize purine bases more strongly than pyrimidine bases [9] . Though the very crucial 162 role of SUD-C has not been recognised in SARS-CoV but in few reports SUD-C has 163 been shown to have RNA-binding ability, and concerted actions by the subdomains of 164 SUD have been proposed to ensure specific RNA binding [9] .The complete deletion of 165 the SUD-C domain within the context of a SARS-CoV was found to have large 166 reduction of RNA synthesis, but some basal RTC activity remains [12] . PDB blast 167 confirmed the presence of mutation in SUD domain of SARS-CoV with 74% identity to 168 2KAF (PDB-ID). Using Swiss model this particular domain was modelled for Reference 169 Wuhan sequence (NC 045521)and the change was observed by insilco protein stability 170 tool DynaMut [15] . The change was predicted to be stabilising by DynaMut and The novel Coronavirus, being identified as a global threat have been the prime focus of 178 research and investigation around the globe. Since its outbreak in Wuhan (China) it has 179 taken up several changes to keep the survival high. There are reports by several groups 180 on mutations observed in the strains from different countries [16] [20] [16] . Here, in this 181 study, the primary focus is on strains submitted by India. The country is suspected to 182 have more number of positive cases than actually reported and also, there is increase in 183 the number of cases despite the lockdown implemented by the government. With cause of this is unclear, but there is a possibility as per previous report that patient 195 \n",
      ". CC-BY-ND 4.0 International license author/funder. It is made available under a The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.14.041301 doi: bioRxiv preprint might have infected multiple times. This same infected person has another notable 196 change of three consecutive positions (28881G>A, 28882G>A, and 28883G>C) 197 reported mainly in the US. To an extent it clears the doubt that person might have 198 contracted multiple strains (Asian and American), thereby carrying both S and L types 199 of strain along with the US reported mutation. L-type was found to be more aggressive 200 strain before Jan 2020 [17] . People who might have contracted this patient would have 201 been at higher risk of infection. In phylogenetic tree analysis these two strains could be 202 observed without branching from the ancestral strain and in vicinity to reference strain 203 sequence (NC 045512) Figure 2 . tourists (Sample 5 to Sample 9), the person (Sample 4) who had contracted with Italian 220 tourist and 6 (Sample 11 to Sample 16) of the Vero CCL81 isolate P1 sequenced in India. 221 It seems to be absent in the strain sequenced from patient with travel history to Italy. 222 One probable explanation to this novel non-synonymous change could be related to 223 its increased transmission rate since it is being observed to be incorporated in 224 combination with already co-existing mutations [20] . The mutation was predicted with 225 stabilising effect, hence no change at secondary structure level was observed Figure 3 .\n",
      "Interaction experiments or insilico docking might help to understand its effect more 227 clearly. Adaptive evolutionary changes in NSP3 gene have been discerned with 228 betacoronaviruses, which might contribute towards the host-range or other viral 229 phenotypes [7] . In a recent study [18] on analysing the possible drug targets through The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.14.041301 doi: bioRxiv preprint 14408C>T, 23403A>G, 4809C>T presented in Figure 4 .\n",
      "As discerned with reported frequent mutations in Europeans, the inference could be 249 made that these mutations could be correlated to efficiency of viral transmission, since 250 Europe has emerged with most adverse effects by the virus. The virus have been 251 reported with slow changes and thereby found to be circulating in few major forms.\n",
      "Looking at the data from Indian strains it has majorily the European strain with the 253 novel change observed in this study. The other subtype observed was the Indian citizens 254 sequenced in Iran. In a phylogenetic tree analysis ( Figure 2 ) it was observed that 255 strains containing this novel mutation were branched together and separate clad of 256 citizens came from Iran.\n",
      "Novel incorporation of one more mutation could be associated with its transmission 258 in Indian population. This change might add up as new member to the group of number 259 of changes the virus has taken in course of evolution in India since the first Italian case 260 identified in Mar 2020. Until now no strain has been declared more lethal in comparison 261 with other, but transmission has been the prime observation, which to an extent could 262 be controlled by social distancing mechanism. This speculation is being made from the 263 very early observations and based on what has been the mutational trend in and around 264 the world. The limitation of this study is the less number of sequences submitted from 265 India. There is an emergent need of more public data sharing from Indian community 266 to track new subtypes of virus, impact of the changes it is incorporating and where the 267 hotspot for therapeutics lies. India is observing the rapid increase in number of cases The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.14.041301 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.14.041301 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.14.041301 doi: bioRxiv preprint\n",
      "Eighteen years ago, severe acute respiratory syndrome (SARS) broke out globally, which caused more than 8000 cases with a fatality rate of 9.6% (1). Since December 2019, a pneumonia infection with 2019-nCoV, which was now named as Novel Coronavirus Pneumonia (NCP), broke out in Wuhan, Hubei Province, China, and rapidly spread throughout China and to many other countries (2) (3) (4) . As of February 9, 2020, 2019-nCoV had transmitted to 34 provinces, regions and municipal cities across China. A total of 40,235 confirmed NCP cases with 908 deaths (2.3%) and 6,484 (16.1%) critically ill cases, and there were still 23,589 suspected cases (5) .\n",
      "As an emerging infectious disease, we need to quickly understand its etiological, epidemiological and clinical characteristics to take prevention and control measures. Several studies have described the epidemiological and clinical characteristics of NCP, and have demonstrated that it can transmitted between humans (4, 6) . A few studies have also estimated the basic reproduction number (R0) of NCP (6) (7) (8) (9) (10) (11) . For instance, Li et al. computed the R0 of 2.2 using daily NCP data before January 4 in Wuhan. Our study using early epidemic stage data also found reproductive number of NCP was 2.9 (11) . However, most of these studies employed daily reporting cases at very early epidemic stage or were purely based on mathematical modeling. It has been suggested that epidemic size may be seriously under-reported due to the unknown etiology and lack of diagnosis protocol at early epidemic stage (12) , which may lead to underestimation of R0. In addition, for control measures to be optimized during ongoing NCP epidemics, temporal changes in reproduction number must be tracked. However, none of previous studies have estimated time-varying instantaneous reproduction number (Rt), which is author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint 4 an important parameter for assessing whether the control and prevention measures are effective or whether additional measures are needed (13) . Moreover, excerpt for Hubei Province, most of confirmed cases reporting from other provinces were imported from Hubei Province at early epidemic stage. With increasing secondary cases in those provinces, however, it is unknown how much the Rt for secondary cases is in those Provinces. Knowing this information is very helpful to assess the epidemic trend and the effects of control measures in those provinces.\n",
      "As a milestone of global public health events, the transmission dynamics of SARS have been well studied (14, 15) . Comparing the transmissibility of NCP with SRAS could imply decision making on the prevention and control of NCP. Therefore, in this study, we described the epidemiological characteristics of NCP using the latest data across China, and estimated the basic and time-varying reproduction numbers which were compared with SARS. Our findings are critical to optimize prevention and control measures during the ongoing epidemic of NCP, as well as guide clinical management. author/funder. All rights reserved. No reuse allowed without permission. (17) and WHO website (18) , respectively.\n",
      "Because only the daily number of reporting cases were obtained from regions out of Guangdong Province, we estimated the daily number of incidences using a generalized additive model (GAM) (Section 1.2 in Supplementary materials). First, we collected each individual onset date and reporting date in Guangdong province which was treated as a sample of all confirmed cases nationwide. Second, a GAM model was used to establish the relationship between onset date and reporting date, and obtained the lagged probability distribution of daily number of incidences for the number of reporting cases. Third, we used the lagged probability distribution author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint 6 to predict the daily number of incidences based on the daily number of reporting cases in other provinces of China, which was added up to obtain the daily incidences nationwide. In order to test the validity of this approach, we randomly divided all cases in Guangdong into two groups (group A and B). Cases in group A was used to establish a GAM model, which was then used to predict the number of incidences in group B. We then estimated the correlation between the actual incidence and the estimated incidence, and the results of cross-validation showed that the R 2 was 89.7%, and the root mean square error (RMSE) was 4.11 ( Figure S1 ).\n",
      "The number of occurred-but-not-yet-reported cases were further estimated during the study period, which caused by reporting delays. Reasons for such delays include the time to complete diagnosis test, logistics, and overwhelmed surveillance systems. To minimize the effects of reporting delay, we applied a Bayesian nowcasting model (Nowcasting by Bayesian Smoothing) to complement the daily total number of incidences (19) . This model is characterized by its ability to deal with the uncertainty in the delay distribution and the time evolution of epidemic curve.\n",
      "In the early phase of an infectious disease outbreak, the initial reproductive rate approximated an exponential growth rate (20) . We selected a period of exponential growth in the epidemic curve to calculated R0, which was defined as the expected number of secondary cases produced by a typical primary case. Since the daily incidence data is an integer, Poisson regression was used to fit the exponential growth rate. We used a prior Gamma distribution for the generation time, with a shape parameter (mean generation time of 7.5 days) and a scale parameter (standard author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint 7 deviation of 3.4 days) from a previous study (7) . We estimated R0 of all confirmed NCP cases nationwide and in Wuhan, and R0 of SARS cases in Guangdong Province, Hong Kong and Beijing. We also estimated R0 for secondary NCP cases in Guangdong Province.\n",
      "In contrast to R0, time-varying instantaneous reproduction number (Rt) represents the average number of secondary cases that would be produced by a typical primary case infected at time t if conditions remained constant after time t (13) . We estimated Rt from the time series of onset cases, with the same Gamma distribution of generation time as R0. In order to maintain the accuracy of the prediction and without hiding the underlying time trend, Rt values were estimated over a 10-day moving window. For the period of estimation, the end date was selected as the latest date of the available data, and the starting date was selected as the earliest date for which the 10-day instantaneous reproduction number estimated above could be assumed constant. We estimated Rt for all confirmed NCP cases nationwide, and in Wuhan, and for secondary NCP cases in Guangdong Province.\n",
      "We applied frequency and percentages (%) to describe categorical variables, and used meanÂ±SD to describe the continuous variables. The doubling time of NCP and SARS were defined and estimated using a method proposed by Galvani et al (21) .\n",
      "A series of sensitivity analyses were conducted to quantify the effect of parameters changes on R0 value. We changed the shape parameter of Gamma distribution from 7.0 to 8.0 days for estimating the R0 of NCP cases, and changed the same parameter from 8.0 to 9.0 for estimating author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint 8 the R0 of SARS cases. R software (version 3.6.0) was used for data analyses, with \"NobBs\" package for building nowcast model, \"R0\" for calculating R0 and \"EpiEstim\" for estimating Rt. Two tailed P<0.05\n",
      "were considered statistically significant for all statistical tests.\n",
      "Data collection and analysis of cases were determined by the National Health Commission of the People's Republic of China to be part of a continuing public health outbreak investigation and were thus considered exempt from institutional review board approval.\n",
      "author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint\n",
      "As of February 7, 2020, a total of 34,598 confirmed cases were identified in 34 provinces, Daily numbers of reporting cases continuously increased to the peak on February 4, 2020, and slightly decreased then after nationwide and in Wuhan. The peak of daily reporting cases was on January 31, 2020 in Guangdong Province, and then significantly decreased. The daily incidence after nowcasting showed similar trend with the reporting data. The doubling time of NCP was 2.4, 2.8, and 3.6 days nationwide, in Wuhan, and in Guangdong Province, respectively (Table 1) The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint prior to the peak, respectively ( Table 1) .\n",
      "The Table 1 ).\n",
      "During the epidemic, at national level, Rt values increased from 6.9 (95%CI: 5.5-8.4) before January 9 to a peak of 8.8 (95%CI: 8.3-9.4) before January 16, and then continuously decreased to 1.59 (95%CI: 1.57-1.61) before February 6. The Rt values in Wuhan continuously decreased from 6.2 (95%CI: 4.9-7.6) by January 9 to 2.10 (95%CI: 2.07-2.14) before February 6. By contrast, Rt values for secondary NCP cases in Guangdong Province was kept at a low-level ranging from 0.8 (95%CI: 0.6-1.1) to 0.2 (95%CI: 0.2-0.3) during the epidemic (Figure 3 ).\n",
      "The results of sensitivity analyses showed the R0 values of NCP and SARS were robust to the changes of GT ( Figure S3 ). For example, the R0 of NCP cases nationwide changed from 4.0 (95%CI: 3.9-4.1) using GT=7.0 to 4.9 (95%CI:4.8-5.1) using GT=8.0. author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint\n",
      "The 2019-nCoV is a novel coronavirus, which is different from SARS and other SARS-like viruses(3). However the time-varying transmission dynamics of NCP during the epidemic remain unclear(22). In this study, we estimated the R0 of NCP infected with 2019-nCoV nationwide, and found that the R0 was 4.5, which means an average of 4.5 secondary cases generated by a primary case in the early stage. An approaching R0 of 4.4 was found in Wuhan.\n",
      "Our estimated R0 was larger than that (R0=2.2) reported by Li et al. who employed the data in Wuhan between December 10, 2019 and January 4, 2020, the very early stage of this epidemic (7) . It was debated that the number of NCP cases might be seriously underreported in the early stage (10, 23) , since 2019-nCoV was confirmed on January 7, and the official diagnosis protocol was released by WHO until January 17 (12) . In addition, the surveillance was not widely conducted, and many infections may be missed. Read et al. estimated that only 5.1% of infections in Wuhan were identified in the early period (23). Zhao et al. estimated that the R0 of NCP was larger than 5.0 using national reported data from January 10 to January 24, 2020, under the scenario that the reporting rate was not changed during that period (12) .\n",
      "Although the fatality rate of NCP cases was much lower than SARS(1), we observed higher R0 and much shorter doubling time of NCP than SARS (14, 24, 25) , indicating the higher transmissibility of 2019-nCoV. These differences can be due to two reasons. First, epidemiological and clinical evidences suggest that asymptomatic or mild NCP cases during their incubation periods could effectively transmit 2019-nCoV, which was different from SARS because most SARS cases were infected by \"super spreaders\", and SARS cases in incubation period and mild cases could not infected susceptible (26) . The average incubation period of author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint NCP was 4.8 days, ranging from 1 to 14 days, and the average period from onset to isolation was 2.9 days (11) broadly conducted through various channels, which has improved the general public to understand the risk of NCP, and take voluntary actions to detect, diagnose and treat cases author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint 13 infected 2019-nCoV earlier. This can be verified by decreasing period from onset to diagnosis during the epidemic in Guangdong Province ( Figure S5 ).\n",
      "We also observed that the R0 was less than 1, and the Rt values continuously declined after January 23 for secondary NCP cases in Guangdong Province, which demonstrates that the implemented prevention and control measures have significantly reduced the risk of indigenous outbreak of NCP, and that the increasing NCP cases were dominated by imported cases.\n",
      "Meanwhile, we found that two secondary cases were infected through sharing the public elevator with cases from their neighbors, which indicates the higher risk of transmission of NCP in community. Similar cases were also reported in other regions. For example, Wang et al.\n",
      "reported that 57 (including 40 healthcare workers and 17 patients) out of 138 hospitalized patients were presumed to have been infected with 2019-nCoV in a Wuhan hospital (29) .\n",
      "Therefore, more efforts are needed to prevent the transmission in communities and other public places.\n",
      "This study has several strengths. First, this is the first study, to date, to assess the time-varying transmission dynamics of NCP based on incidence data nationwide. We also estimated instantaneous reproduction number for secondary cases in Guangdong Province. Secondly, we compared the transmission dynamics of NCP with SARS using the same methods. Third, we initially assessed the effects of control measures previously taken by the governments. These findings could provide important information for optimizing control measures.\n",
      "Our study has several limitations. First, the incidence date of confirmed cases in provinces except for Guangdong Province were estimated by a GAM modeling, which may lead to misclassification bias. However, the cross-validation test indicated a good performance of our author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint 14 GAM modeling. In addition, the R0 (3.8, 95%CI: 3.4-4.2) estimated for imported cases in Guangdong Province, as a sample of the total confirmed cases nationwide, was comparable with the R0 estimated nationwide, indicating that our results are robust. Second, we are still in the early stage of this outbreak, and there is much uncertainty in epidemiological parameters that affects transmissibility of NCP. Sequential studies are urgently needed to fill-in these knowledge gaps.\n",
      "In summary, NCP may have a higher pandemic risk than SARS in 2003, and the efforts of containing the outbreak are taking into effect. Our findings indicate that rigorous control and prevention measures taken by Chinese governments on early detection, diagnosis and treatment of NCP cases are still needed to uphold until the time-varying reproduction number below one. author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint\n",
      "The data that support the findings of this study are available from the corresponding author on reasonable request. Participant data without names and identifiers will be made available after approval from the corresponding author. After publication of study findings, the data will be available for others to request. The research team will provide an email address for communication once the data are approved to be shared with others. The proposal with detailed description of study objectives and statistical analysis plan will be needed for evaluation of the reasonability to request for our data. The corresponding author will make a decision based on these materials. Additional materials may also be required during the process.\n",
      "author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint *: The R value was estimated in secondary cases in Guangdong Province.\n",
      "author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint 22 Figure 1 . Distribution of NCP cases worldwide and in China, as of February 7, 2020\n",
      "Panel A: The spatial distribution of NCP cases worldwide.\n",
      "Panel B: The spatial distribution of NCP cases in China. author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.01.25.919787 doi: bioRxiv preprint\n",
      "On January 21, 2020, the first case of Coronavirus disease 2019 in the 2 United States was reported in Washington State [1] . The first case was not reported in 3 New York state until March 1, 2020 [2] . By the time the World Health Organization 4 (WHO) declared a global pandemic on March 11, 2020 , there were 345 cases in New 5 York City (NYC), and this number skyrocketed to nearly 18,000 cases just two weeks 6 later [2, 3] . NYC rapidly became the epicenter of the pandemic in the United States, 7 with a transmission rate five times higher than the rest of the country, and over a third 8 of all confirmed national cases by early April [4] . 9 During a pandemic, there is likely to be large variation in both disease transmission 10 and disease testing between regions [5] . These two factors cause large variation in 11 disease reporting between different areas [6] . This is particularly true in the early stages 12 of the outbreak, before disease testing has become widespread and standardized. 13 Contemporary and historical studies on previous pandemics, including H1N1 14 pandemics in 1918 and 2009, suggest that socioeconomic factors on a national level can 15 affect detection rates and medical outcomes [7] [8] [9] . Thus, socioeconomic factors such as 16 young or old populations, race, affluence, inequality, poverty, unemployment, insurance, 17 or access to healthcare may account for differences in reported cases of COVID-19 18 between neighborhoods in NYC. 19 The aim of this ecological study was to identify potential neighborhood-level 20 socioeconomic determinants of the number of detected COVID-19 cases and explain 21 between-neighborhood variation during the early, exponential growth stage of the 22 pandemic in NYC: from the first detected case in March 1 until April 5, 2020. 23 Materials and methods 24 Data collection 25 Data on positive COVID-19 cases were collected from NYC Department of Health and 26 Mental Hygiene (DOHMH) Incident Command System for COVID-19 Response 27 (Surveillance and Epidemiology Branch in collaboration with Public Information Office 28 Branch) [2] . Since the NYC DOHMH was discouraging people with mild to moderate 29 symptoms from being tested during the time period covered, the data primarily 30 represents people with more severe illness. Since at the time of writing the pandemic is 31 still ongoing, data were taken at a snapshot on April 5, 2020. This date was chosen to 32 cover the first month of the pandemic in NYC, since understanding early etiology of the 33 pandemic and local influences is important in helping to inform future management [10] . 34 Data were a cumulative count up to and including April 5, 2020. On this date, NYC 35 had a cumulative total of 64,955 cases [11] , including deaths and hospitalizations. 36 The available dataset included 64,512 cases (99.3% of total cases), with each case 37 representing a positive diagnosis of COVID-19 along with the patient's Zip Code 38 Tabulation Area (ZCTA). ZCTAs are generalized areal representations of United States 39 Postal Service (USPS) Zip Code service areas. ZCTAs were the areas in which patients 40 reported their home address, as opposed to either where they became symptomatic or 41 where they reported for testing/treatment. The area of interest covered 177 ZCTAs Queens, and Staten Island); Fig 1B displays Bureau American Community Survey (ACS). ACS is a continuous sample survey of 52 3.5 million households every year including questions beyond the decadal census on 53 subjects such as education, employment, internet access, and transportation. Data were 54 collected at ZCTA level from the ACS 2014-2018 5-year estimate [12] , which is the most 55 recent publicly available. The 5-year estimate was chosen instead of the most recent 56 1-year estimate because the latter was not available in an aggregated form at ZCTA 57 level. Whilst the 5-year estimate is less current, it has a smaller margin of error than 58 the 1-year estimates and greater statistical reliability for small geographic areas. i.e., populations typically economically inactive. Aged population was the percentage of 65 the total population 65+. The increased severity of COVID-19 with increasing age has 66 been well documented [13] , and there has been recent evidence of asymptomatic carrier 67 transmission particularly amongst young people [14, 15] . Males per 100 females was 68 chosen to capture the balance of sex in the population. We were interested in whether 69 sex differences lead to significant variation in detected cases. Finally, some reports 70 suggest a racial disparity in case detection rates across the USA. A report from NYU 71 Furman Center for housing, neighborhoods, and urban policy suggests mortality rates 72 are higher amongst the city's \"Hispanic, Black, and non-Hispanic/Latino: Other\" 73 populations [16] . For the present study, we chose to include the percentage of the against the adverse effects of unequal income [17] , an association has been demonstrated 85 between inequality and population health [18] . We also included household income, 86 which was a significant predictor for hospitalizations in the 2009 influenza 87 pandemic [19] . Specifically, in the present study we use median household income as a 88 ZCTA-level predictor. Finally, unemployment and poverty both have documented 89 association with health outcomes, including in pandemic scenarios [20, 21] . Whilst there 90 is some level of collinearity between these two variables, we include both as one relates 91 to the economically active labor force whereas the other to the total population. The \n",
      "Two parameters related to healthcare access were included in the study: percentage of 95 population uninsured, Uninsured; and total number of hospital bed per 1000 people 96 within 5 km, Beds. It has been documented that lack of insurance can delay access to 97 timely healthcare, particularly during pandemics [22] . We hypothesized that this could 98 affect virus transmission and/or access to testing, therefore affecting detection rates.\n",
      "Finally, we chose Beds as a parameter related to proximity to healthcare, which has 100 been shown to be inversely associated with adverse outcomes in other geospatial public 101 health studies [23] . For a city containing multiple hospitals such as NYC, we defined a 102 proximity metric in this study as population normalized number of hospital beds within Prior to analysis of potential predictors, we considered multiple base regression models. 110 Given the significant spatial correlation in the present case data as evidenced by the 111 Moran Index, I (176) = 0.481, p < 0.0005 [24] , we explored potential regression models 112 both with and without spatial effects. We compared four base models (no predictors): 113 1) a Poisson model with random intercept; 2) a Poisson Besag-York-MolliÃ© (BYM) by Eq 1:\n",
      "where Ï… i has an intrinsic conditional autoregressive (ICAR) structure [27] . We used the 119 reparameterization of the BYM model proposed by Riebler et al. [28] , known as the 120 BYM2 model and shown in Eq 2:\n",
      "where Ï„ Î³ is the overall precision hyperparameter, Ï• âˆˆ [0, 1] is the mixing hyperparameter 122 representing the proportional division of variance between the spatial and nonspatial 123 effects, Ï… * is the spatial (ICAR) effect with a scaling factor such that Var (Ï… * ) â‰ˆ 1, and 124 Î½ * is the nonspatial random-effect with Î½ * âˆ¼ N (0, 1). Penalized complexity (PC) priors 125 are applied to hyperparameters Ï„ Î³ and Ï• (compared to log-gamma priors in the random 126 intercept model) [29] . All four models used ZCTA population as the exposure and a 127 log-link function. We selected the model with the lowest Deviance Information Criterion 128 (DIC) [30] , representing the best trade-off between model fit and complexity.\n",
      "Characteristics for the four base models examined, including hyperparameters, are 130 shown in \n",
      "i , scaled nonspatial random-effect; Ï… * i , scaled spatial random-effect with intrinsic conditional autoregressive structure; Ï„ Î½ , precision for nonspatial random effect, log-Gamma prior; Ï„ Î³ , overall precision, penalized complexity (PC) prior; Ï•, mixing parameter, PC prior; n, overdispersion parameter, PC Gamma prior.\n",
      "Multiple regression models were built using a method adjusted from Nikolopoulos et 136 al. [31] . In the univariable models, we considered each predictor variable separately (i.e., 137 one model per variable). In the multivariable model we considered all predictor 138 variables together. We further built a partial multivariable model using only those 139 predictors that were significant in the univariable models. Finally we built a model 140 using stepwise backwards elimination procedure, starting with the fully saturated model 141 April 16, 2020 5/16\n",
      "All rights reserved. No reuse allowed without permission.\n",
      "author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.04.17.20069823 doi: medRxiv preprint and removing the least significant predictor until we were left with a model containing 142 only significant predictors [31] . In all cases, the expected number of detected COVID-19 143 cases in ZCTA i, Î» i , was represented by Eq 3:\n",
      "where E i is the exposure (i.e., population) for ZCTA i, Î² 0 is the intercept, Î² p is 145 coefficient of the fixed effect for predictor p âˆˆ {1...P }, x ip is the value of predictor p in 146 ZCTA i, and the spatial and nonspatial random effects for ZCTA i are described by the 147 BYM2 model detailed above. Vague Gaussian priors are assumed on all Î². between income, poverty, and unemployment. There was only one bivariate correlation 178 above 0.7 (median household income and poverty), and none above 0.8. It was decided 179 to leave all predictors in the analysis and to build multiple regression models in order to 180 consider the effects of collinearity. Fig 6 shows author/funder, who has granted medRxiv a license to display the preprint in perpetuity. Males per 100 females. 4 Percentage of population that identify as white (alone or in combination with another race). 5 Gini index. 6 Median household income in $1,000s. 7 Percentage of working age population unemployed. 8 Percentage of population living below the poverty threshold. 9 Percentage of population uninsured. population, and median household income as predictors. Table 3 shows a summary of 215 the regression estimates from this model. Fig 7A shows the area specific relative risk Î¶ i 216 for this model, whilst Fig 7B shows the posterior probability that the relative risk is 217 greater than 1, p (Î¶ i > 1|y). In this model, a 1% increase in the dependent population 218 leads to a 2.5% (95% CI: 1.6% to 3.4%, p < 0.0005) increase in detected COVID-19 219 cases. An increase of 1 male per 100 females leads to a 1.0% (95% CI: 0.6% to 1.5%, 220 p < 0.0005) increase in detected cases. Finally, a $10,000 decrease in median household 221 income leads to a 2.5% (95% CI: 1.0% to 4.1%, p = 0.002) increases in detected cases. 222 proportion of dependents in the population and male to female ratio. We also see a 236 significant inverse relationship between detected cases and median household income. 237 We do not see a consistently significant relationship between detected cases and the 238 other potential predictors; even those such as poverty, unemployment, and proximity to 239 healthcare that were significant in a univariable model.\n",
      "Our findings indicated statistically significant associations between two of the four 241 demographic predictors included in the study. We find percentage of dependents in the 242 population to be a statistically significant predictor in all of the models in which it 243 appears as a factor. Conversely, we find that the aged percentage of the population 244 (65+) is never a significant predictor of detected COVID-19 cases. We believe that 245 taken together, this indicates that the youthful end of the dependent population is 246 associated with higher case numbers. This is congruent with evidence from Chan et 247 al. [14] and Bai et al. [15] , both of whom suggest significant transmission by young 248 asymptomatic carriers. We further hypothesize that attitudes and behavioral patterns 249 could play a significant role in this effect. As an example, increasing mortality of 250 COVID-19 with age has been well publicized, and we suggest this may incline older 251 communities to adhere to preventative public-health measures more. Conversely the 252 same information may be interpreted by younger populations that they are not at 253 significant risk, potentially encouraging riskier behaviors.\n",
      "The balance of males and females as a significant factor indicates that areas with Studies have posited sex differences in immunological function [36] or smoking 260 prevalence/pattern [37] as potential causes of differing medical outcomes. We found no 261 studies to date examining sex specific behavior trends in relation to COVID-19 262 transmission and incidence. Looking back further, we found conflicting evidence from 263 studies on the 2009 H1N1 pandemic. Some studies suggested that females were more 264 willing to engage in public health precautions [38] , whilst others suggested no significant 265 sex effects [39] . We suggest that further studies be undertaken to consider whether sex 266 specific behavioral, employment, or other trends are mechanisms that could explain sex 267 effects on detected cases. Race was marginally insignificant in our statistical analysis.\n",
      "Thus, we caution against any association beyond the confounding sociological 269 relationship between race and economic affluence [40] . Further, our study used 270 percentage of population identifying as white, splitting the demographic into two 271 groups, white and non-white. A study exploring multiple racial categories at an 272 aggregate and/or individual level, may draw additional insight [19] .\n",
      "Regarding the economic predictors, we note that our findings are in agreement with 274 a previous, non-pandemic study [41] , which found that affluence (in our case household 275 income) was a significant predictor on self-rated health whilst poverty and income 276 inequality (the Gini index) were not significant factors. Wen et al. suggest that the 277 presence of affluence sustains neighborhood social organizations, which in turn positively 278 affect health. If we extend this argument to the current pandemic, we could hypothesize 279 that these social organizations further act to pass on information and promote 280 community adoption of transmission-reduction policies such as social distancing [42] .\n",
      "Furthermore, we note that those in low affluence neighborhoods are more likely to live 282 in higher density residence arrangements, for example community housing and shared 283 family dwellings. The increased urban density likely contributes to transmission of the 284 virus among the neighborhood [43] . Whilst previous studies [44] have found influence of 285 unemployment on disease transmission, we note that the unprecedented shut-down of 286 national infrastructure and the economy has meant that many previously employed 287 people suddenly found themselves either unemployed, furloughed, or working from home. 288 In a short period of time this drastic measure has completely altered the employment 289 landscape of NYC such that it is unsurprising the predictor from 2018 is not significant. 290 We found that neither of our healthcare-related predictors was consistently 291 significant. Lack of insurance has previously been a barrier to both diagnosis and 292 treatment [45, 46] . However, in the COVID-19 pandemic, significant state resources were 293 directed such that testing was freely available to all eligible New York residents. 294 We note four key limitations of the ecological study. First, our dependent variable is 295 the number of detected COVID-19 cases, which may be significantly different from the 296 number of true cases [47] . We believe, however that this does not detract from the 297 validity of the study, since characterization of the detection and prevalance is important 298 for pandemic management [48] . Studies on HIV rates amongst at risk populations 299 suggest that the relationship between predictors and the number of detected cases is 300 likely a complex interaction via at least three pathways: the true number of cases, access 301 to testing (means) [49] , and population attitudes to testing (motivation) [50, 51] . Thus, 302 we can still develop valid inferences, even if we cannot elicit with certainty which one (or 303 ones) of these pathways the significant predictors act through. Second, any associations 304 made must be interpreted with caution since, as with any observational study, spurious 305 correlations produced by unstudied confounding factors may be present. Caution is also 306 advised due to the ecological fallacy of making individual inferences from aggregate data. 307 Further verification is required to determine true causative links between predictors and 308 detected cases even when associations are significant. Third, the significant predictors 309 found are likely not the only explanations for different detection rates between different 310 neighborhoods. However this study does provide useful insight into explaining city limits at the borough level, there may be borough-level biases relating to 313 COVID-19 testing. However, if these biases exist, they likely inhibit testing access in 314 low-income neighborhoods [52, 53] such that the inverse association found between 315 income and positive cases is more pronounced than what the model suggests.\n",
      "Within the constraints imposed by the limitations of an ecological analysis, we conclude 318 that there exist consistent, significant associations between COVID-19 detected cases 319 and the percentage of dependents in the population and male to female ratio. Further, 320 there is also a significant association between detected COVID-19 cases and low income 321 neighborhoods. The significance of dependents likely comes from the youth rather than 322 the aged population. We suggest further studies be undertaken to determine any 323 underlying causative mechanisms to these associations, paying particular attention to 324 willingness to engage in public health behaviors and to asymptomatic carrier 325 transmission. We finally highlight that whilst predictors may change with increased 326 time and access to testing, this study provides important insights into public health 327 behavior in the early stages of the current, and future, pandemic. The authors declare that the research was conducted in the absence of any commercial 333 or financial relationships that could be construed as a potential conflict of interest. The datasets analyzed for this study are publicly available, a repository can be found on 341 GitHub: https://github.com/rswhittle/NYC-COVID19-socioeconomic. \n",
      "Facioscapulohumeral muscular dystrophy (FSHD) affects over 25,000 people in the U.S. alone, making it one of the most prevalent genetic diseases. The genetic mutation underlying FSHD is usually a reduction in the copy number of a macrosatellite repeat on chromosome 4 referred to as D4Z4 (van Deutekom et al. 1993; Wijmenga et al. 1992 ). This repeat is GC-rich, highly methylated, and normally subjected to repeat-induced silencing, which is disrupted in an allelespecific manner by contractions to 10 or fewer copies (van Overveld et al. 2003) or on all D4Z4 repeats by mutation in the chromatin protein SMCHD1 (de Greef et al. 2009; Hartweck et al. 2013 ; Lemmers et al. 2012) . When silencing at D4Z4 breaks down, an RNA transcript encoding the DUX4 protein (Gabriels et al. 1999 ) is expressed. The presence of a poly(A) signal downstream of the D4Z4 repeats on chr4 (Dixit et al. 2007 ) leads to DUX4 expression and explains why disease is associated only with contractions of the D4Z4 repeats on chr4, and only on specific 4qter alleles, so called permissive alleles, which harbor the poly(A) signal (Lemmers et al. 2010; Lemmers et al. 2004 ). The DUX4 protein has been observed by immunostaining of cultured FSHD myoblasts which show infrequent, possibly episodic, expression (Jones et al. 2012; Snider et al. 2010 ) and differentiated myotubes which show more prevalent expression (Block et al. 2013; Rickard, Petek and Miller 2015) .\n",
      "Mechanisms of DUX4-mediated pathology are currently being investigated actively. At high levels of expression, DUX4 causes toxicity that leads to cell death of myoblasts (Bosnakovski et al. 2008; Kowaljow et al. 2007 ), while at low levels of expression, it impairs myogenic differentiation (Dandapat et al. 2014) and sensitizes cells to oxidative stress (Bosnakovski et al. 2008 ). A mouse model allowing doxycycline-regulated DUX4 expression has recapitulated these phenotypes in primary myoblasts (Dandapat et al. 2014) while also having several nonmuscle related phenotypes due to low basal levels of DUX4 expression in the absence of All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/118315 doi: bioRxiv preprint doxycycline (Dandapat et al. 2016 ). Other animal model work includes a mouse carrying human D4Z4 repeats (Krom et al. 2013) which showed some evidence of sporadic DUX4 expression, but no myopathy; and adeno-associated viral (AAV) vector-mediated delivery of DUX4 to skeletal muscle, which showed profound myopathy (Wallace et al. 2011 ). This latter work implicated the p53 pathway in DUX4 pathology, as the p53 knockout background suppressed AAV-DUX4 toxicity. The linkage to p53 is compelling, as this pathway has the potential to push cells into apoptosis, however precisely how DUX4 would activate p53 is not clear. The immediate targets of the DUX4 transcription factor include genes with regulatory elements containing the sequence TAATCTAATCA (Geng et al. 2012; Zhang et al. 2015) , or variants thereof. ChIP-seq has identified many genomic targets (Choi et al. 2016; Geng et al. 2012 ) and at the majority of these, DUX4 displaces nucleosomes, recruits p300 and/or CBP through its C-terminus which promotes acetylation of histone H3 and activation of transcription (Choi et al. 2016 ).\n",
      "Here, we directly test the p53-dependence of the pathogenic phenotypes of DUX4 expression, by inhibiting p53 in cell lines while DUX4 is expressed, and by crossing the iDUX4[2.7] transgene onto the p53 knockout background. These experiments reveal that p53 is not necessary for DUX4-mediated pathology, neither in myoblasts, nor in other tissues.\n",
      "Previously, overexpression of Pax3 or Pax7 was been shown to reduce the toxicity of DUX4 to C2C12 mouse myoblasts (Bosnakovski et al. 2008) . To see to what extent inhibiting p53 could similarly reduce DUX4 toxicity, we transduced iC2C12-DUX4 cells, which are immortalized mouse myoblasts engineered for doxycycline-inducible DUX4 expression, with retroviral vectors expressing several constructs known to interfere with p53: a dominant negative p53 (R175H All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/118315 doi: bioRxiv preprint mutation), MDM2 (Momand et al. 1992) , and TRIM24 (Allton et al. 2009 ). Overexpressing cell lines were established and cells were exposed to two different doses of doxycycline, both of which resulted in significant cell killing in the absence of overexpression (empty vector control).\n",
      "Notably, although the positive controls, Pax3 and Pax7, protected substantially at 24 hours, cells transduced with the three p53 inhibitors did not show a reduced death rate compared to empty vector (Fig. 1 ). This indicates that in this system, p53 is not a relevant player in the cell death phenotype.\n",
      "We have previously generated a mouse model with a doxycycline-inducible 2.7 kb DUX4 transgene (Dandapat et al. 2014; Dandapat et al. 2016) , referred to as iDUX4[2.7]. One of the interesting features of this mouse is that the very low basal level of expression from the Tet-on promoter in the absence of doxycycline leads to various non-muscle phenotypes, especially in males, where 80% die as embryos, with the remaining 20% being severely runted and all dying before 6 weeks of age. Females are less severely affected and can thus propagate the strainbecause the transgene is X-linked, X-inactivation diminishes the phenotype in females. We reasoned that if p53 were necessary for the pathological effects of DUX4 on embryonic cell types, then on a p53 knockout background, males ought to be born at normal ratios, and ought to be relatively healthy compared to siblings with a functional copy of p53. We therefore crossed the iDUX4[2.7] transgene onto the p53 knockout background. Female carriers were obtained that were homozygous for the p53 knockout and these were bred to male p53 heterozygotes. We genotyped 65 progeny from this backcross and obtained no DUX4+ males, neither on the p53 knockout nor on the heterozygous background (Fig. 2) . Thus, absence of p53 has no effect on the pathological effects of DUX4 on development in the iDUX4[2.7] strain.\n",
      "Although no males were produced from the backcross described above, we did eventually obtain live born p53 iDUX4[2.7] knockout males from F1 crosses to Rosa-rtTA p53 double All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/118315 doi: bioRxiv preprint heterozygous males, and these rare animals displayed the severe runting and skin phenotypes (flaky skin, alopecia, and puffy eyelids) typical of the iDUX4[2.7] mouse (Dandapat et al. 2014 ).\n",
      "Thus p53 is not necessary for the pathological effects of DUX4 on non-muscle tissues.\n",
      "To determine whether DUX4 would be cytotoxic to muscle cells in the absence of p53, we established primary cell cultures from muscle tissue of these iDUX4[2.7] ; p53 KO animals.\n",
      "Primary muscle cells were sorted into myogenic and fibro-adipogenic fractions by flow cytometry for VCAM/Itga7 and PDGFRÎ±, respectively. These sorted primary cultures were then exposed to doxycycline over a series of doses to induce DUX4 expression to different levels. DUX4 expression was clearly cytopathic (Fig. 3A) and caused a dose-dependent loss of cellular viability in myoblasts (Fig. 3B) . The same was observed for fibroadipogenic progenitors (Fig.   3B ) on the p53 knockout background. Thus, in the absence of p53, DUX4 is still cytotoxic, to both myogenic and fibroadipogenic progenitors.\n",
      "Using both the inducible C2C12 cell culture system, where it was originally demonstrated that DUX4 was pathogenic to myogenic progenitors (Bosnakovski et al. 2008) , and the iDUX4[2.7] mouse model (Dandapat et al. 2014 ), we find no evidence for p53 involvement in DUX4 pathogenicity. Interestingly, this applies to phenotypes caused by very low basal levels of DUX4 expression, as the developmental effects and runting are thought to be, as well as overt cytotoxicity in primary or immortalized cells, which is the result of high level DUX4 expression.\n",
      "The previous study implicating p53 in DUX4 toxicity made use of a very different experimental system with vastly higher levels of DUX4 expression: recombinant adeno-associated virus vector-mediated production. The levels of DUX4 expression achieved in a cell infected with multiple copies of DUX4 driven from the CMV promoter are much higher than those in which DUX4 is expressed from its endogenous promoter from a single (terminal) D4Z4 repeat in All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/118315 doi: bioRxiv preprint FSHD. It is possible that at these exceptionally high levels of expression, the p53 pathway does in fact become necessary for some aspect of extreme pathology, however the relevance of such high levels of expression is debatable. The nature of the muscle damage seen in the AAV model is quite distinct from that seen in FSHD, and DUX4 expression is in fact quite difficult to detect in biopsies of FSHD muscle, to the point where the most reliable readout of its expression in biopsies is its target gene fingerprint (Yao et al. 2014 ). It is also theoretically possible that some aspect of the AAV experimental system combines with the effect of DUX4 to make the pathology more severe than it would otherwise have been, and the p53 effect ascribed to DUX4 was in fact downstream of the cellular response to AAV infection.\n",
      "Nevertheless, the studies presented here clearly demonstrate that p53 is not necessary for cytotoxicity of DUX4.\n",
      "Animal work was conducted under a protocol approved by the University of Minnesota IACUC.\n",
      "The p53 mutant allele used in these studies was obtained from Jackson Labs (strain 008651) and carries a floxed stop codon upstream of a dominant negative (R270H) p53 mutant. In the absence of cre, no functional p53 is produced, thus as used in the current study, it is a null allele. Mice homozygous for this allele developed papillomas and most eventually succumbed to thymic lymphomas, as predicted for a p53 null.\n",
      "The iC2C12-DUX4 cell line was cultured in high glucose Dulbecco's Modified Eagle Media (DMEM) supplemented with 20% fetal bovine serum (FBS, Atlanta Biologicals), L-glutamine and sodium pyruvate (Gibco), penicillin and streptomycin (P/S, Gibco) at 37 Â°C in 5% CO2. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/118315 doi: bioRxiv preprint Primary mouse myoblasts and muscle fibroblasts were isolated and expanded as previously described (Arpke et al. 2013; Dandapat et al. 2014) . Briefly, hind limb muscles were dissected under sterile conditions, minced using razor blade and digest with collagenase type II (Gibco;\n",
      "Grand Island, NY, 17101-015) and dispase (Gibco, 17105-041) . Single cell suspensions were plated in T75 dishes in F-10/Ham's (Hyclone) medium containing 20% FBS (HyClone), 50 ng Î¼ L -1 human basic fibroblast growth factor (Peprotech), 1% penicillin/streptomycin (Gibco), and 1% Glutamax (Gibco) and cultured at 37Â°C at 5% O 2 , 5% CO 2 , for 5 days. Establishing overexpressing cell lines Retroviral expression constructs were co-transfected with the packaging constructs pCL-Eco (Naviaux et al. 1996) into NIH-3T3 cells using FuGENE 6 transfection reagent (Roche). Viruscontaining supernatants were collected 48 hours later, filtered (0.45 Î¼ m) and used directly to transduce iC2C12-DUX4 cells. Constitutively overexpressing cell lines were obtained by FACS sorting GFP+ cells. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https: //doi.org/10.1101/118315 doi: bioRxiv preprint Cell death inhibition assays Cell viability was analyzed as previously described . Briefly, myoblasts and fibroblasts were plated in 96 well plates at 3000 cells per well in proliferation medium and treated with different concentrations of doxycycline for 72 h. Medium was removed from the plate; 100 ÂµL working (diluted 1:3 with PBS) CellTiter-Glo luminescent assay reagent (Promega) was added to each well and luminescence measured on a Cytation3 plate reader (BioTek). The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/118315 doi: bioRxiv preprint exposed to doxycycline (dox) to induce DUX4 expression (below). Scale bar represents 100 Î¼m.\n",
      "(B) Viability assay for the same cells, exposed to different doses of doxycycline. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/118315 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. All rights reserved. No reuse allowed without permission.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/118315 doi: bioRxiv preprint The copyright holder for this preprint (which was not peer-reviewed) is the author/funder. . https://doi.org/10.1101/118315 doi: bioRxiv preprint\n",
      "Pharyngeal virus shedding was very high during the first week of symptoms (peak at 7.11 X 10 8 RNA copies per throat swab, day 4). Infectious virus was readily isolated from throat-and lung-derived samples, but not from stool samples in spite of high virus RNA concentration. Blood and urine never yielded virus. Active replication in the throat was confirmed by viral replicative RNA intermediates in throat samples. Sequence-distinct virus populations were consistently detected in throat-and lung samples of one same patient. Shedding of viral RNA from sputum outlasted the end of symptoms. Seroconversion occurred after 6-12 days, but was not followed by a rapid decline of viral loads. COVID-19 can present as a mild upper respiratory tract illness.\n",
      "There is a close genetic relatedness between severe acute respiratory syndrome coronavirus (SARS-CoV) and the causative agent of COVID-19, SARS-CoV-2. The predominant expression of ACE2 in the lower respiratory tract is believed to have determined the natural history of SARS as a lower respiratory tract infection. Whereas positive SARS-CoV-2 detection in clinical specimens from the upper respiratory tract has been described 9 , these observations do not address principal differences between SARS and COVID-19 in terms of clinical pathology. The here-studied patients were enrolled because they acquired their infections upon known close contact to an index case, thereby avoiding representational biases due to symptom-based case definitions. All patients were treated in a single hospital in Munich, Germany. Virological testing was done by two closely-collaborating laboratories using the same standards of technology for RT-PCR and virus isolation, confirming each other's results based on almost all individual samples. Due to extremely high congruence of results, all data are presented together. Only the serological data are based on results from one laboratory. The patients are part of a larger cluster of epidemiologically-linked cases that occurred after January 23 rd , 2020 in Munich, Germany, as discovered on January 27 th (BÃ¶hmer et al., accompanying manuscript) . The present study uses samples taken during the clinical course in the hospital, as well as from initial diagnostic testing before admission. In cases when this initial diagnostic testing was done by other laboratories, the original samples were retrieved and re-tested under the rigorous quality standards of the present study.\n",
      "To first understand whether the described clinical presentations are solely caused by SARS-CoV-2 infection, samples from all patients were tested against a panel of typical agents of respiratory viral infection, including HCoV-HKU1, -OC43, -NL63, -229E; Influenza virus A and B, Rhinovirus, Enterovirus, Respiratory syncytial virus, Human Parainfluenza virus 1-4, Human metapneumovirus, Adenovirus, and Human bocavirus. Interestingly, no co-infection was detected in any patient.\n",
      "All patients were initially diagnosed by RT-PCR 10 from oro-or nasopharyngeal swab specimens. Both specimen types were collected over the whole clinical course in all patients.\n",
      "There were no discernible differences in viral loads or detection rates when comparing nasovs. oropharyngeal swabs ( Figure 1B) . The earliest swabs were taken on day 1 of symptoms, with symptoms often being very mild or prodromal. All swabs from all patients taken between days 1 and 5 tested positive. The average virus RNA load was 6.76x10 5 copies per whole swab until day 5 (maximum, 7.11X10 8 copies/swab). Swab samples taken after day 5 had an average viral load of 5.13x10 3 copies per swab and a detection rate of 45.95%. The last swab sample was taken on day 22 post-onset. Average viral load in sputum was 1.18 x 10 6 copies per mL (maximum, 6.65x10 8 copies per mL).\n",
      "Because swab samples had limited sensitivity for initial diagnosis of cases of SARS 13,14 , we analyzed the first paired swab and sputum samples taken on the same occasion from seven patients ( Table 1) . All samples were taken between 2 and 4 days post-onset. In two cases, swab samples had clearly higher virus concentrations than sputum samples, as indicated by a difference greater than 3 in threshold cycle (Ct) value. The opposite was true in two others cases, while the other 5 cases had similar concentrations in both sample types.\n",
      "None of 27 urine samples and none of 31 serum samples were tested positive for SARS-CoV2 RNA.\n",
      "To understand infectivity, live virus isolation was attempted on multiple occasions from clinical samples (Figure 1 D) . Whereas virus was readily isolated during the first week of symptoms from a significant fraction of samples (16.66% in swabs, 83.33% in sputum samples), no isolates were obtained from samples taken after day 8 in spite of ongoing high viral loads.\n",
      "Virus isolation from stool samples was never successful, irrespective of viral RNA concentration, based on a total of 13 samples taken between days six to twelve from four patients. Virus isolation success also depended on viral load: samples containing <10 6 copies/mL (or copies per sample) never yielded an isolate. For swab and sputum, interpolation based on a probit model was done to obtain laboratory-based infectivity criteria for discharge of patients (Figures 1 E, F) .\n",
      "High viral loads and successful isolation from early throat swabs suggested potential virus replication in upper respiratory tract tissues. To obtain proof of active virus replication in absence of histopathology, we conducted RT-PCR tests to identify viral subgenomic messenger RNAs (sgRNA) directly in clinical samples. Viral sgRNA is only transcribed in infected cells and is not packaged into virions, therefore indicating the presence of activelyinfected cells in samples. Viral sgRNA was compared against viral genomic RNA in the same sample. In sputum samples taken on days 4/5, 6/7, and 8/9, a time in which active replication in sputum was obvious in all patients as per longitudinal viral load courses (see below), mean normalized sgRNA per genome ratios were ~0.4% ( Figure 1G ). Swabs taken up to day 5 were in the same range, while no sgRNA was detectable in swabs thereafter. Together, these data indicate active replication of SARS-CoV-2 in the throat during the first 5 days after symptoms onset. No, or only minimal, indication of replication in stool was obtained by the same method ( Figure 1G ).\n",
      "During our study we sequenced full virus genomes from all patients. A G6446A exchange was first detected in one patient and later transmitted to other patients in the cluster (BÃ¶hmer, accompanying manuscript). In the first patient, this mutation was found in a throat swab while a sputum sample from the same day still showed the original allele, 6446G. The SNP was analyzed by RT-PCR and Sanger sequencing in all sequential samples available from that patient ( Table 1) . The presence of separate genotypes in throat swabs and sputum strongly supported our suspicion of independent virus replication in the throat, rather than passive shedding there from the lung.\n",
      "Daily measurements of viral load in sputum, pharyngeal swabs, and stool are summarized in Figure 2 . In general, viral RNA concentrations were very high in initial samples. In all patients except one, throat swab RNA concentrations seemed to be already on the decline at the time of first presentation. Sputum RNA concentrations declined more slowly, with a peak during the first week of symptoms in three of eight patients. Stool RNA concentrations were also high. Courses of viral RNA concentration in stool seemed to reflect courses in sputum in many cases (e.g., Figure 2 A, B, C) . In only one case, independent replication in the intestinal tract seemed obvious from the course of stool RNA excretion (Figure 2 D) .\n",
      "Whereas symptoms mostly waned until the end of the first week ( Table 2) , viral RNA remained detectable in throat swabs well into the second week. Stool and sputum samples remained RNA-positive over even longer periods, in spite of full resolution of symptoms. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 /2020 All cases had comparatively mild courses ( Table 2 ). The two patients who showed some signs of pneumonia were the only cases where sputum viral loads showed a late and high peak around day 10/11, whereas sputum viral loads were on the decline by this time in all other patients (Figure 2 F,G) .\n",
      "Seroconversion was detected by IgG and IgM immunofluorescence using cells expressing the spike protein of SARS-CoV-2 and a virus neutralization assay using SARS-CoV-2 ( Table   3 ). In early sera, taken between day 3 and 6, none of the patients showed detectable antibody. The patients monitored long enough to yield a serum sample after two weeks all showed neutralizing antibodies, the titer levels of which did not suggest any correlation with clinical courses. Of note, case #4, with the lowest virus neutralization titer at end of week 2, seemed to shed virus from stool over prolonged time (Figure 2 D) . Results on differential recombinant immunofluorescence assay indicated no significant rise in titer against the four endemic human Coronaviruses (Table S1) .\n",
      "The clinical courses in subjects under study were mild, all being young-to middle-aged professionals without significant underlying disease. Apart from one patient, all cases were first tested when symptoms were still mild or in the prodromal stage, a period in which most patients would present once there is general awareness of a circulating pandemic disease 5 .\n",
      "Diagnostic testing suggests that simple throat swabs will provide sufficient sensitivity at this stage of infection. This is in stark contrast to SARS. For instance, only 38 of 98 nasal or nasopharyngeal swab samples tested positive by RT-PCR in SARS patients in Hong Kong 15 .\n",
      "Also, viral load differed considerably. In SARS, it took 7 to 10 days after onset until peak RNA concentrations (of up to 5x10 5 copies per swab) were reached 13,14 . In the present study, peak concentrations were reached before day 5, and were more than 1000 times higher.\n",
      "Successful live virus isolation from throat swabs is another striking difference from SARS, for which such isolation was rarely successful [16] [17] [18] . Altogether, this suggests active virus replication in upper respiratory tract tissues, where only minimal ACE-2 expression is found and SARS-CoV is therefore not thought to replicate 19 . At the same time, the concurrent use of ACE-2 as a receptor by SARS-CoV and SARS-CoV-2 corresponds to a highly similar excretion kinetic in sputum, with active replication in the lung. SARS-CoV was found in sputum at mean concentrations of 1.2-2.8x10 6 copies per mL, which corresponds to observations made here 13 . Whereas proof of replication by histopathology is awaited, extended tissue tropism of SARS-CoV-2 with replication in the throat is strongly supported by our studies of sgRNAtranscribing cells in throat swab samples, particularly during the first 5 days of symptoms.\n",
      "Striking additional evidence for independent replication in the throat is provided by sequence findings in one patient who consistently showed a distinct virus in her throat as opposed to the lung. Critically, the majority of patients in the present study seemed to be already beyond One of the most interesting hypotheses to explain a potential extension of tropism to the throat is the presence of a polybasic furin-type cleavage site at the S1-S2 junction in the SARS-CoV-2 spike protein that is not present in SARS-CoV 17 . Insertion of a polybasic cleavage site in the S1-S2 region in SARS-CoV was shown to lead to a moderate but discernible gain of fusion activity that might result in increased viral entry in tissues with low density of ACE2 expression 20 .\n",
      "The combination of very high virus RNA concentrations and occasional detection of sgRNAcontaining cells in stool indicate active replication in the gastrointestinal tract. Active replication is also suggested by a much higher detection rate as compared to MERScoronavirus, for which we found stool-associated RNA in only 14.6% samples in 37 patients hospitalized in Riyadh, Saudi Arabia 21,22 . If virus was only passively present in stool, such as after swallowing respiratory secretions, similar detection rates as for MERS-CoV would be expected. Replication in the gastrointestinal tract is also supported by analogy with SARS-CoV, which was regularly excreted in stool, from which it could be isolated in cell culture 23 .\n",
      "Our failure to isolate live SARS-CoV-2 from stool may be due to the mild courses of cases, with only one case showing intermittent diarrhea. In China, diarrhea has been seen in only 2 of 99 cases 24 . Further studies should therefore address whether SARS-CoV-2 shed in stool is rendered non-infectious though contact with the gut environment. Our initial results suggest that measures to contain viral spread should aim at droplet-, rather than fomitebased transmission. All rights reserved. No reuse allowed without permission.\n",
      "author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the .\n",
      "The prolonged viral shedding in sputum is relevant not only for hospital infection control, but also for discharge management. In a situation characterized by limited capacity of hospital beds in infectious diseases wards, there is pressure for early discharge following treatment.\n",
      "Based on the present findings, early discharge with ensuing home isolation could be chosen for patients who are beyond day 10 of symptoms with less than 100,000 viral RNA copies per ml of sputum. Both criteria predict that there is little residual risk of infectivity, based on cell culture.\n",
      "The serological courses of all patients suggest a timing of seroconversion similar to or slightly earlier than in SARS-CoV infection 18 . Seroconversion in most cases of SARS occurred during the second week of symptoms. As in SARS and MERS, IgM was not detected significantly earlier than IgG in immunofluorescence, which might in part be due to technical reasons as the higher avidity of IgG antibodies outcompetes IgM for viral epitopes in the assay. IgG depletion can only partially alleviate this effect. Because IFA is a laborintensive method, ELISA tests should be developed as a screening test. Neutralization testing is necessary to rule out cross-reactive antibodies directed against endemic human coronaviruses. Based on frequently low neutralizing antibody titers observed in coronavirus infection 12,25 , we have here developed a particularly sensitive plaque reduction neutralization assay. Considering the titers observed, a simpler microneutralization test format is likely to provide sufficient sensitivity in routine application and population studies.\n",
      "When aligned to viral load courses, it seems there is no abrupt virus elimination at the time of seroconversion. Rather, seroconversion early in week 2 coincides with a slow but steady decline of sputum viral load. Whether certain properties such as glycosylation pattern at critical sites of the glycoprotein play a role in the attenuation of neutralizing antibody response needs further clarification. In any case, vaccine approaches targeting mainly the induction of antibody responses should aim to induce particularly strong antibody responses in order to be effective. All rights reserved. No reuse allowed without permission.\n",
      "author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the I  n  f  l  u  e  n  z  a  a  n  d  S  A  R  S  -c  o  r  o  n  a  v  i  r  u  s  a  c  t  i  v  a  t  i  n  g  p  r  o  t  e  a  s  e  s  T  M  P  R  S  S  2  a  n  d  H  A  T  a  r  e  e  x  p  r  e  s  s  e  d  a  t  m  u  l  t  i  p  l  e  s  i  t  e  s  i  n  h  u  m  a  n  r  e  s  p  i  r  a  t  o  r  y  a  n  d  g  a  s  t  r  o  i  n  t  e  s  t  i  n  a  l  t  r  a  c  t  s  .   P  l  o  S  o  n  e   7  ,  e  3  5  8  7  6  ,  d  o  i  :  1  0  .  1  3  7  1  /  j  o  u  r  n  a  l  .  p  o  n  e  .  0  0  3  5  8  7  6  (  2  0  1  2  )  .  2  0  B  e  l  o  u  z  a  r  d  ,  S  .  ,  C  h  u  ,  V  .  C  .  &  W  h  i  t  t  a  k  e  r  ,  G  .  R  .  A  c  t  i  v  a  t  i  o  n  o  f  t  h  e  S  A  R  S  c  o  r  o  n  a  v  i  r  u  s  s  p  i  k  e  p  r  o  t  e  i  n  v  i  a  s  e  q  u  e  n  t  i  a  l  p  r  o  t  e  o  l  y  t  i  c  c  l  e  a  v  a  g  e  a  t  t  w  o  d  i  s  t  i  n  c  t  s  i  t  e All rights reserved. No reuse allowed without permission.\n",
      "author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.20030502 doi: medRxiv preprint isolation success based on probit distributions. The inner lines are probit curves (doseresponse rule). The outer dotted lines are 95% CI. For less than 5% isolation success, the estimated day was 9.78 (95% CI: 8.45-21.78) days post-onset and the estimated RNA concentration for less than 5% isolation success was estimate to be 6.51 Log10 RNA/ml (95% CI:-4,11-5.40). H, Subgenomic viral RNA transcripts in relation to viral genomic RNA. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.03.05.20030502 doi: medRxiv preprint author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 /2020 \n",
      "Sputum-and stool samples were taken and shipped in native condition. Oro-and nasopharyngeal throat swabs were preserved in 3 mL of viral transport medium. Viral loads in sputum samples were projected to RNA copies per mL, in stool to copies per g, and in throat swabs to copies per 3 mL, assuming that all sample components were suspended in the 3 mL viral transport medium. For swab samples suspended in less than 3 mL viral transport medium, this conversion was adapted to represent copies per whole swab. An aggregated overview of samples received per day post onset of disease from all patients is shown in Figure 1A .\n",
      "RT-PCR used targets in the E-and RdRp genes as described 10 . Both laboratories used a pre-formulated oligonucleotide mixture (Tib-Molbiol, Berlin, Germany) to make laboratory procedures more reproducible. All patients were also tested for other respiratory viruses, \n",
      "Virus isolation was done in two laboratories on Vero E6 cells. 100 Âµl of suspended, cleared, and filtered clinical sample was mixed with an equal volume of cell culture medium.\n",
      "Supernatant was harvested after 0, 1, 3, and 5 days and used in RT-PCR analysis.\n",
      "Additional technical details are provided in Section 2a in the Supplementary Appendix.\n",
      "We performed recombinant immunofluorescence assays to determine the specific reactivity against recombinant spike proteins in VeroB4 cells, as described 11,12 . This assay used cloned CoV spike protein from HCoV-229E, HCoV-NL63, HCoV-OC43, HCoV-HKU1, and SARS-CoV-2. The screening dilution was 1:10. Plaque reduction neutralization tests were done essentially as previously described for MERS-CoV 12 . Serum dilutions causing plaque reductions of 90% (PRNT90) and 50% (PRNT50) were recorded as titers. Additional technical details are provided in Section 2b and 2c in the Supplementary Appendix.\n",
      "All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 /2020 Statistical analyses were done using SPSS software (Version 25) or GrapPad Prism (Version 8).\n",
      "All patients provided informed consent to the use of their data and clinical samples for the purposes of the present study. Institutional review board clearance for the scientific use of patient data has been granted to the treating institution by the Ethikkommission bei der Medizinischen FakultÃ¤t der Ludwig Maximillians UniverstitÃ¤t MÃ¼nchen. All rights reserved. No reuse allowed without permission. author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10. 1101 /2020 \n",
      "(2.2 with 90% high density interval 1.4-3.8 (Riou and Althaus, 2020) or 2.7 with a 95% CrI of 2.47-2.86 (Wu et al., 2020) ). Higher estimates have also been reported (Sanche et al., 2020; Yang et al., 2020) . Importantly, R 0 is not a biological constant for a pathogen, but is affected by factors such as environmental conditions and the behaviour of infected individuals. One such environmental factor is climate which modulates transmissibility throughout the year. As a result, many respiratory viruses show clear seasonal variation in prevalence; the most well-known example being seasonal influenza which peaks every winter in the temperate zone of the Northern Hemisphere (Petrova and Russell, 2018) . A similar pattern is seen for the four seasonal human coronaviruses: HKU1, NL63, OC43 and 229E (hereafter collectively referred to as \"seasonal CoVs\") (Al-Khannaq et al., 2016; Friedman et al., 2018; Galanti et al., 2019; GÃ³es et al., 2019; Huang et al., 2017; Killerby et al., 2018) . These viruses cause respiratory infections which usually are mild and primarily affect young children.\n",
      "Previous influenza pandemics have swept the world in multiple waves often but not always coinciding with winter months in temperate climates (Amato-Gauci et al., 2011; Taubenberger et al., 2019; Viboud et al., , 2016 . The 1968 The -1970 global influenza pandemic was sparked by a new influenza A/H3N2 virus, with a divergent hemagglutinin protein, which replaced the A/H2N2 virus circulating for 10 years previously . The virus spread rapidly, but the viral dynamics and mortality was not synchronized across countries. While many experienced two more-severe flu seasons in the winters of 1969 and 1970, the US had higher mortality in the first season, while European countries, Japan, and Australia, had higher mortality in the second, and Canada had roughly equal mortality in both .\n",
      "The 2009 pandemic H1N1 virus (A/H1N1pdm09) originated in March 2009 in Mexico and spread around the globe within weeks. Only a few European countries saw substantial circulation of H1N1pdm09 in the spring of 2009. Instead the virus showed low prevalence over the summer and pronounced peaks in the following autumn and winter in many countries (Amato-Gauci et al., 2011) . A/H1N1pdm09 has subsequently transitioned into a seasonal pattern causing winter epidemics in temperate climates.\n",
      "Here we use data on seasonal variation in prevalence of seasonal CoVs in Sweden and model the impact of this variation on the possible future spread of SARS-CoV-2 in the temperate zone of the Northern Hemisphere. We also explore different scenarios of SARS-CoV-2 spread in temperate and tropical regions and show how variation in epidemiological parameters affects a potential pandemic and the possibility of transitioning to an endemic state.\n",
      "Data on seasonal variation of HKU1, NL63, OC43 and 229E diagnoses in respiratory samples was obtained from the routine molecular diagnostics at the Karolinska University Hospital, Stockholm, Sweden. The laboratory provides diagnostic services to six of seven major hospitals and approximately half of outpatient care in the Stockholm county (2.2 million inhabitants). We extracted pseudonymized data on all analyses for the four viruses between Jan 1, 2010 and Dec 31, 2019. The dataset included a total of 52,158 patient samples with 190,257 diagnostic tests, of which 2,084 were positive for any of the coronaviruses (229E = 319; NL63 = 499; OC43 = 604; HKU1 = 355; OC43/HKU1 = 307). Metadata included information about date of sampling and age of patient. In the period of Jan 1, 2010 to Nov 5, 2017, the coronavirus diagnostic was done using in-house assays (Tiveljung-Lindell et al., 2009) . From Nov 6, 2017 to Dec 31, 2019, samples were analysed using the commercial kit Allplex Respiratory Panels (Seegene Inc., Seoul (South Korea)). This commercial kit does not distinguish between HKU1 and OC43, and for this reason positive tests for these two viruses were combined for the entire study period.\n",
      "The fraction of tests that were positive for the four seasonal CoVs showed a strong and consistent seasonal variation, see Fig. 1 . From December to April approx-imately 2% of tests were positive, while less than 0.2% of tests were positive between July to September, i.e. a 10-fold difference (Fig. 1, right) . The strength of variation of the transmission rate through the year could be of high relevance to the spread of SARS-CoV-2 in 2020 and following years.\n",
      "We consider simple SIR models (Kermack and McKendrick, 1991) with an additional category E of exposed individuals of the form\n",
      "where Î²(t) is the rate at which an infected individual infects a susceptible one, Âµ is the inverse incubation time, Î½ is the recovery rate and b is the population turn-over rate.\n",
      "Depending on the analysis below, we implement several such populations that exchange individuals through migration, for details see Supplementary Methods. Stochasticity is implemented through Poisson resampling of the population once every serial interval Âµ âˆ’1 + (Î½ + b) âˆ’1 . The population turnover b rate is immaterial for a pandemic scenario, but important for our analysis of seasonal CoVs, and should be interpreted as the sum of the birth rate and the rate at which previously immune individuals become susceptible due to immune waning and escape. We review general properties of such model in the Supplementary Materials. Following previous work, we parameterize transmissibility as\n",
      "where Î² 0 is the average annual infection rate, Îµ is the amplitude of seasonal forcing which modulates transmissibility through the year, and Î¸ is the time of peak transmissibility (Chen and Epureanu, 2017; Dushoff et al., 2004) . For simulations of the pandemic, we will add an additional term to Î²(t) that accounts for infection control measures in heavily affected areas, see Supplementary Materials.\n",
      "Seasonal CoVs are endemic throughout the world and we therefore expect that viruses are imported throughout the year. We model this import through migration of susceptible individuals with rate m that return exposed . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint with probability x. Humans develop immune responses to CoVs rapidly and subsequent challenge studies show reduced susceptibility and less severe disease for a year (Callow et al., 1990) . Antibodies against SARS-CoV-1 persist for several years (Guo et al., 2020) . This is consistent with the observation that about 50% of all positive samples in our data come from patients older than 10 years with a flat distribution across age groups. In analogy to the attack rate of seasonal influenza, we assume humans suffer from a seasonal CoV infection on average every 10 years (b = 0.1/y). Furthermore, we use R 0 = 2.3, a recovery rate of 0.2days âˆ’1 , and an incubation period of 5 days.\n",
      "With these assumptions, we can solve the model and compare the resulting trajectories to the seasonal variation in prevalence of seasonal CoVs, see Fig. 2 . In Stockholm, seasonal variation of CoVs (especially HKU1/OC43) is very consistent across years (see Fig. S3 ). We therefore fit the SIER model to the average seasonal variation across years by calculating the squared deviation of observed and predicted prevalence relative to their respective mean values. Simulations of the model are compatible with observations in two separate regions of parameter space: If Northern Europe was very isolated with less than 1 in 1,000 susceptible individuals returning with a seasonal CoV infection from abroad each year, weak seasonality of around Îµ = 0.15 would be sufficient to generate strong variation through the year compatible with observations (Fig. 2 , bottom-left ridge). In this regime, prevalence is oscillating intrinsically with a period that is commensurate with annual seasonal oscillations giving rise to a resonance phenomenon with annual or biennial patterns even for weak seasonal forcing (Chen and Epureanu, 2017; Dushoff et al., 2004) .\n",
      "If the rate of import of seasonal CoV infections is higher, imports dampen the resonance and much stronger seasonality, with values between Îµ = 0.3 and 0.7, is required to fit the observations (Fig. 2 , top-right-and-center ridge). In this regime, seasonal variation in transmissibility modulates the size of micro-outbreaks triggered by imported cases in a mostly immune population.\n",
      "These two scenarios differ slightly in the time of year at which peak transmissibility Î¸ occurs: When transmission is mostly local and seasonality is amplified by . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 resonance, Î¸ needs to be around October-November to fit the data with most cases in December-January. In the second scenario with high connectivity, Î¸ needs to be in December-January coinciding with the peak in prevalence. Given that most countries are highly connected, we focus here on exploring the high-import and strong seasonal forcing scenario. This scenario, with maximal Î² in mid-winter, is also more compatible with climate variation around the year. The qualitative behavior of the fit is robust to uncertainty in R 0 and the frequency of reinfection b.\n",
      "The analysis of seasonal CoV prevalence patterns allowed us to constrain parameter ranges and explore different scenarios of SARS-CoV-2 spread around the globe, in particular in temperate climates like Northern Europe.\n",
      "Here we explore scenarios where temperate regions have a seasonal forcing of between Îµ = 0.3 and 0.7 and migration rates of 0.01/year. Early estimates suggest an incubation time of about 5 days and an average serial interval of 7-8 days (Wu et al., 2020) . Our model uses an average incubation time of 5 days (Backer et al., 2020) and an infectious period of 5 days.\n",
      "To match the R 0 estimates for the early outbreak with our parameterization of transmissibility in Eq. 2 we need to account for the fact that December/January are winter months in Hubei and peak transmissibility in Hubei likely corresponds to Î¸ â‰ˆ 0 (0 being the beginning of the year, so a Î¸ in December/January). An R 0 â‰ˆ 3 in winter in Hubei and a seasonal forcing of Îµ = 0.4 implies an annual average R 0 = Î² 0 /Î½ = 2.2. This reasoning leads to our parameter choice of Î² 0 = 158/year, Î½ = 72/year, Î¸ = 0. We assume the outbreak started at t = 2019.8 in Hubei with one infected individual and use N = 6 Ã— 10 7 as population size. To incorporate infection control measures, transmissibility is reduced by 50% once prevalence reached 3% (third order Hill-function, see Supplemental Material). Introductions to a location like Northern Europe with Îµ = 0.5 (i.e. slightly stronger seasonal forcing then Hubei) are assumed to happen at a rate of 0.01 per year for each infected individual elsewhere. The simulation of the SIER model in different regions is deterministic, but migration is implemented stochastically by Poisson resampling of the average number of migrating individuals. Fig. 3 shows simulated trajectories of SARS-CoV-2 prevalence in the temperate Northern Hemisphere assuming the outbreak started in Hubei early December 2019. Depending whether the peak transmissibility of SARS-CoV-2 in the northern temperate zone is in November, January, or March, the simulation predicts a main peak in the first half of 2020, a main peak in winter 2020/2021, or two similarly sized peaks.\n",
      "To explore possible scenarios more systematically, we ran such simulations for a range of values for R 0 and peak transmissibility Î¸ and recorded whether we observe and early peak, a late peak, or a two peaks. The right panel of Fig. 3 shows the ratio of the height of these peaks for different values. Rapid growth (high R 0 ) and late transmission peaks result in a large peak in the first half 2020, while lower R 0 and transmission peaks in early winter favor a large secondary peak. These two scenarios are separated by a band of parameter values that give rise to two pandemic waves in the winters of 2020 and 2021 in the Northern Hemisphere. Individual trajectories for a variety of parameter combinations are given in Fig. S4 . The qualitative behavior is robust to model perturbations and parameter variation as long as seasonal forcing is strong. With weak forcing (Îµ = 0.15), the model predicts a single peak for most combinations of R 0 and migration rates (see Fig. S5 ).\n",
      "The uncertainty in parameter values and the potential impact of infection control measures imply that all scenarios are plausible and should be considered when developing pandemic prevention and containment strategies.\n",
      "In absence of control measures, outbreaks initially grow exponentially within well-mixed communities, and at a certain rate the virus will be carried to other regions and potentially seed new outbreaks. Such export to new locations is initially unlikely, but becomes next to certain once the outbreak size exceeds the inverse probability that a given individual migrates while infected. We have witnessed such rapid dispersal to SARS-CoV-2 to many countries across the globe during January and February 2020.\n",
      "Every location has a different socio-economic profile such that the growth rate of the epidemic (and hence R 0 ) might differ. The superposition of many such subpopulations with a range of R 0 values and seasonal variation in transmission will result in dynamics that are qualitatively different from a single population SIR model. In particular, such variation result in a pandemic spread out over 2 years before the virus possibly becomes endemic. Fig. 4 shows the result of such a simulation of 1,000 populations. Populations were divided between northern temperate (50%), southern temperate (10%), and tropical (40%) and assigned parameters as follows:\n",
      "â€¢ R 0 was drawn from a normal distribution with mean 2.2 (see Supplementary Fig. 7 for R 0 = 1.5 and 3.0) and standard deviation 0.5.\n",
      "â€¢ Seasonal forcing Îµ was drawn from a uniform distribution between 0.25 and 0.75 for temperate regions, . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https: //doi.org/10.1101 //doi.org/10. /2020 between 0 and 0.2 for tropical regions.\n",
      "â€¢ Peak transmissibility Î¸ of temperate regions was drawn from normal distributions with standard deviation 0.1 and peak at 0 for northern regions and 0.5 for southern regions. Î¸ for tropical regions was chosen uniformly from between 0 and 1.\n",
      "â€¢ Population sizes were drawn from a log-normal distribution with Ïƒ = 1 and a mean such that all populations sum to 7.6 billion.\n",
      "â€¢ Migration rates were sampled from a log-normal distribution with Ïƒ = 1 and a mean of 0.01.\n",
      "For Hubei, we use the same parameters as described in section IV. The variation in R 0 and migration rate result in a super-position of fast and slow epidemics seeded at different times. The initial phase is dominated by fast epidemics driving rapid dispersal, in particular in the tropics, while slow epidemics dominate later in 2020 and 2021. With the parameter setting used in Fig. 4 , the Northern temperate regions see most circulation in winter 2020/2021. In accordance with Fig. 3 , this peak shifts more towards early 2020 for higher R 0 , see Supplementary Fig. 7 .\n",
      "After several years, SARS-CoV-2 could become a seasonal CoV with characteristic winter outbreaks as shown in Fig. 1 . Such a scenario is demonstrated in Fig. 5 where a simulation similar to the one shown in Fig. 4 is run for 12 years, with the added assumption that after infection an individual become susceptible to SARS-CoV-2 again at a rate of 0.1 per year as we assumed for seasonal CoV above. After a pronounced low in 2020-2024, prevalence recovers and settles into a seasonal pattern, similar to that of the four existing seasonal CoVs.\n",
      "We report on the possible influence of seasonal variation on the spread of SARS-CoV-2 in the Northern Hemisphere, in a pandemic scenario. We find that seasonal variation in transmissibility has the potential to modulate the spread of SARS-CoV-2 with a wide range of possible outcomes that need to be taken into account when interpreting case counts and projecting the outbreak dynamics. The onset of spring and summer could, for example, give the impression that SARS-CoV-2 has been successfully contained, only for infections to increase again in 2020-2021 winter season. Even in Hubei virus circulation might decrease due to containment measures and the arrival of spring but might increase again towards the end of the year. Whether a pandemic in the temperate regions of the Northern Hemisphere would peak early in 2020, late in 2020, or show multiple waves as H1N1pdm did in 2009, depends on the timing of peak transmissibility and the rate of spread (R 0 and serial interval).\n",
      "This study is meant as an exploration of how such a . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https: //doi.org/10.1101 //doi.org/10. /2020 If previously infected individuals can be reinfected after some time, as for example by seasonal influenza virus, SARS-CoV-2 could develop into a seasonal CoV that returns every winter. This would typically happen at much lower prevalence than peak pandemic levels. These simulations assume reinfection on average every 10 years.\n",
      "pandemic could unfold, not as a prediction of any particular scenario. The results we present are critically dependent on the assumptions i) that the outbreak will develop into a pandemic, ii) that the transmissibility of SARS-CoV-2 shows seasonal variability of sufficient strength (range Îµ = 0.3 to 0.7), and iii) that parameters like R 0 estimated from the early phase of the outbreak are comparable in other populations. These assumptions are not implausible but not certain: cases of SARS-CoV-2 has been in several countries in Asia apart from China, as well as in Europe, Africa, North America, and Australia (WHO Emergency Committee, 2020b), and mild or asymptomatic cases make detection and thus prevention of spread by isolation challenging, e.g. airport screening as a preventive measure is unlikely to prevent spread and local seeding (Quilty et al., 2020) . Person-to-person transmission of the virus has been documented in several countries outside of China, including large outbreaks in Iran, South Korea, and Italy (European Centre for Disease Prevention and Control (ECDC), 2020). It is likely that not all exported cases have been detected, and some may have seeded outbreaks outside of China that have yet to be detected.\n",
      "The seasonal CoVs show a strong and consistent seasonal variation, and modeling suggests that this requires strong variation in transmissibility throughout the year. It should be noted, however, that SARS-CoV-2 does seem to transmit in tropical climates like Singapore, and so winter is not a necessary condition of SARS-CoV-2 spread. Furthermore, our models are compatible with work by Luo et al. (2020) showing that recent trends in different regions across East-Asia imply that seasonality alone is unlikely to end SARS-CoV-2 spread. Precise values for the underlying model parameters and the effect of infection control measures are currently unavailable. For this reason we explored a range of parameter values to assess the robustness of the results to model assumptions.\n",
      "The simulations presented here are scenarios that emerge from simplified abstract models, but they nevertheless demonstrate that a wide variety of outcomes are . CC-BY-NC 4.0 International license It is made available under a author/funder, who has granted medRxiv a license to display the preprint in perpetuity.\n",
      "is the (which was not peer-reviewed) The copyright holder for this preprint . https://doi.org/10. 1101 compatible within the limits of the current knowledge about the outbreak. The implications of our work are that: 1) reductions in prevalence need not be attributable to successful interventions, but could be due to seasonal variation in transmissibility, 2) sub-population dynamics can differ greatly, meaning that case count trajectories in one country should be used cautiously to inform projections in a second country, even in the same climate zone, 3) seasonal variation might slow down a pandemic and thereby provide a window of opportunity for better preparation of health care systems world-wide by scaling up capacity for care and diagnostics, and potentially through rapid development of antivirals and vaccine, and 4) after several years SARS-CoV-2 could develop into an endemic seasonal CoV similar to the transition of the 2009 A/H1N1 pandemic influenza virus into a seasonal influenza virus.\n",
      "The overall impact of a potential SARS-CoV-2 pandemic depends critically on the case fatality ratio (CFR), which we have not modelled here. At present, uncertainty in the CFR is high due to likely over-representation of severe cases in the statistics and a delay between diagnosis and recovery/death (Battegay et al., 2020) . Even with this unknown, seasonal variation in transmissibility of SARS-CoV-2 and underlying differences in migration, introduction times, and attack rate should thus be taken into account when monitoring and projecting global transmission, planning further surveillance of the epidemic, and developing pandemic prevention and containment strategies.\n",
      "All relevant data and script that generate the graphs are available in a dedicated github repository at github. com/neherlab/CoV_seasonality.\n"
     ]
    }
   ],
   "source": [
    " for l in df_covid[\"body_text\"][:10]:\n",
    "    print (l)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 4. Run all dataset "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
